
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.aa42804411e1.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.6c8346f5860b.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3b12438b8d66.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.6539a0a78536cfdc1fa6.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.e622fd8870cb011febf1.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>The Protein Folding Problem - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C1ED92F5712D500003D4277478AAC.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.5.0.post1+55721b3" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC2443096/" />


    


        <script>
            
            var useOfficialGovtHeader = true;
        </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2443096/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Annual review of biophysics" /><meta name="citation_title" content="The Protein Folding Problem" /><meta name="citation_author" content="Ken A. Dill" /><meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, University of California, San Francisco, California 94143" /><meta name="citation_author_institution" content="Graduate Group in Biophysics, University of California, San Francisco, California 94143; email: ude.fscu.llewxam@llid" /><meta name="citation_author" content="S. Banu Ozkan" /><meta name="citation_author_institution" content="Department of Physics, Arizona State University, Tempe, Arizona 85287; email: ude.usa@nakzo.unab" /><meta name="citation_author" content="M. Scott Shell" /><meta name="citation_author_institution" content="Department of Chemical Engineering, University of California, Santa Barbara, California 93106; email: ude.bscu.gnireenigne@llehs" /><meta name="citation_author" content="Thomas R. Weikl" /><meta name="citation_author_institution" content="Max Planck Institute of Colloids and Interfaces, Department of Theory and Bio-Systems, 14424 Potsdam, Germany; email: ed.gpm.gkipm@lkiew.samoht" /><meta name="citation_publication_date" content="2008/06/06" /><meta name="citation_volume" content="37" /><meta name="citation_firstpage" content="289" /><meta name="citation_doi" content="10.1146/annurev.biophys.37.092707.153558" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2443096/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC2443096/" /><meta name="citation_pmid" content="18573083" /><meta name="DC.Title" content="The Protein Folding Problem" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Ken A. Dill" /><meta name="DC.Contributor" content="S. Banu Ozkan" /><meta name="DC.Contributor" content="M. Scott Shell" /><meta name="DC.Contributor" content="Thomas R. Weikl" /><meta name="DC.Date" content="2008 Jun 9" /><meta name="DC.Identifier" content="10.1146/annurev.biophys.37.092707.153558" /><meta name="DC.Language" content="en" /><meta property="og:title" content="The Protein Folding Problem" /><meta property="og:type" content="article" /><meta property="og:description" content="The &#x0201c;protein folding problem&#x0201d; consists of three closely related puzzles: (a) What is the folding code? (b) What is the folding mechanism? (c) Can we predict the native structure of a protein from its amino acid sequence? Once regarded ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2443096/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
    


    
        <section class="pmc-alerts">

</section>
    

    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2443096
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC2443096/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC2443096/pdf/nihms51627.pdf" class="int-view">PDF (2.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2443096/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2443096/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<div class="collections-button-container" data-article-id="2443096" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="zAMJXfQ8hT1ArsbvQwKKC3kepYqjt3YQLgbn6cO4aGwPhbwc6daa7ccNiFr4VvGL">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F&amp;text=The%20Protein%20Folding%20Problem" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2443096/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/18573083/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/18573083/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2443096/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC2443096
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="_blank" rel="noopener noreferrer" shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">Annu Rev Biophys.</span> Author manuscript; available in PMC 2008 Jul 3.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18573083" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Annu Rev Biophys. 2008 Jun 9; 37: 289–316. </a></div> <span class="doi"><span>doi: </span><a href="//doi.org/10.1146%2Fannurev.biophys.37.092707.153558" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1146/annurev.biophys.37.092707.153558</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2443096</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS51627</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/18573083">18573083</a></div></div></div><h1 class="content-title">The Protein Folding Problem</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dill%20KA%5BAuthor%5D" class="affpopup" co-rid="_co_idm139787311590544" co-class="co-affbox">Ken A. Dill</a>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ozkan%20SB%5BAuthor%5D" class="affpopup" co-rid="_co_idm139787296921920" co-class="co-affbox">S. Banu Ozkan</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shell%20MS%5BAuthor%5D" class="affpopup" co-rid="_co_idm139787293165472" co-class="co-affbox">M. Scott Shell</a>,<sup>4</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weikl%20TR%5BAuthor%5D" class="affpopup" co-rid="_co_idm139787349329280" co-class="co-affbox">Thomas R. Weikl</a><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139787311590544"><h3 class="no_margin">Ken A. Dill</h3><p><sup>1</sup> Department of Pharmaceutical Chemistry, University of California, San Francisco, California 94143</p><p><sup>2</sup> Graduate Group in Biophysics, University of California, San Francisco, California 94143; email: <a href="mailto:dev@null" data-email="ude.fscu.llewxam@llid" class="oemail">ude.fscu.llewxam@llid</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dill%20KA%5BAuthor%5D">Ken A. Dill</a></div></div><div id="_co_idm139787296921920"><h3 class="no_margin">S. Banu Ozkan</h3><p><sup>3</sup> Department of Physics, Arizona State University, Tempe, Arizona 85287; email: <a href="mailto:dev@null" data-email="ude.usa@nakzo.unab" class="oemail">ude.usa@nakzo.unab</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ozkan%20SB%5BAuthor%5D">S. Banu Ozkan</a></div></div><div id="_co_idm139787293165472"><h3 class="no_margin">M. Scott Shell</h3><p><sup>4</sup> Department of Chemical Engineering, University of California, Santa Barbara, California 93106; email: <a href="mailto:dev@null" data-email="ude.bscu.gnireenigne@llehs" class="oemail">ude.bscu.gnireenigne@llehs</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shell%20MS%5BAuthor%5D">M. Scott Shell</a></div></div><div id="_co_idm139787349329280"><h3 class="no_margin">Thomas R. Weikl</h3><p><sup>5</sup> Max Planck Institute of Colloids and Interfaces, Department of Theory and Bio-Systems, 14424 Potsdam, Germany; email: <a href="mailto:dev@null" data-email="ed.gpm.gkipm@lkiew.samoht" class="oemail">ed.gpm.gkipm@lkiew.samoht</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weikl%20TR%5BAuthor%5D">Thomas R. Weikl</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139787297160032_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139787297160032_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139787297160032_ai" style="display:none"><div class="fm-affl" id="A1"><sup>1</sup> Department of Pharmaceutical Chemistry, University of California, San Francisco, California 94143</div><div class="fm-affl" id="A2"><sup>2</sup> Graduate Group in Biophysics, University of California, San Francisco, California 94143; email: <a href="mailto:dev@null" data-email="ude.fscu.llewxam@llid" class="oemail">ude.fscu.llewxam@llid</a></div><div class="fm-affl" id="A3"><sup>3</sup> Department of Physics, Arizona State University, Tempe, Arizona 85287; email: <a href="mailto:dev@null" data-email="ude.usa@nakzo.unab" class="oemail">ude.usa@nakzo.unab</a></div><div class="fm-affl" id="A4"><sup>4</sup> Department of Chemical Engineering, University of California, Santa Barbara, California 93106; email: <a href="mailto:dev@null" data-email="ude.bscu.gnireenigne@llehs" class="oemail">ude.bscu.gnireenigne@llehs</a></div><div class="fm-affl" id="A5"><sup>5</sup> Max Planck Institute of Colloids and Interfaces, Department of Theory and Bio-Systems, 14424 Potsdam, Germany; email: <a href="mailto:dev@null" data-email="ed.gpm.gkipm@lkiew.samoht" class="oemail">ed.gpm.gkipm@lkiew.samoht</a></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm139787297160032_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18573083" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Annu Rev Biophys</a></div></div></div><div class="sec"></div><div id="abstract-a.l.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.l.b.rtitle">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The &#x0201c;protein folding problem&#x0201d; consists of three closely related puzzles: (<em>a</em>) What is the folding code? (<em>b</em>) What is the folding mechanism? (<em>c</em>) Can we predict the native structure of a protein from its amino acid sequence? Once regarded as a grand challenge, protein folding has seen great progress in recent years. Now, foldable proteins and nonbiological polymers are being designed routinely and moving toward successful applications. The structures of small proteins are now often well predicted by computer methods. And, there is now a testable explanation for how a protein can fold so quickly: A protein solves its large global optimization problem as a series of smaller local optimization problems, growing and assembling the native structure from peptide fragments, local structures first.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">structure prediction, funnel energy landscapes, CASP, folding code, folding kinetics</span></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">INTRODUCTION</h2><p id="P2" class="p p-first-last">The protein folding problem is the question of how a protein&#x02019;s amino acid sequence dictates its three-dimensional atomic structure. The notion of a folding &#x0201c;problem&#x0201d; first emerged around 1960, with the appearance of the first atomic-resolution protein structures. Some form of internal crystalline regularity was previously expected (<a href="#R117" rid="R117" class=" bibr popnode">117</a>), and &#x003b1;-helices had been anticipated by Linus Pauling and colleagues (<a href="#R180" rid="R180" class=" bibr popnode">180</a>, <a href="#R181" rid="R181" class=" bibr popnode">181</a>), but the first protein structures&#x02014;of the globins&#x02014;had helices that were packed together in unexpected irregular ways. Since then, the protein folding problem has come to be regarded as three different problems: (<em>a</em>) the folding code: the thermodynamic question of what balance of interatomic forces dictates the structure of the protein, for a given amino acid sequence; (<em>b</em>) protein structure prediction: the computational problem of how to predict a protein&#x02019;s native structure from its amino acid sequence; and (<em>c</em>) the folding process: the kinetics question of what routes or pathways some proteins use to fold so quickly. We focus here only on soluble proteins and not on fibrous or membrane proteins.</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">THE FOLDING CODE: WHAT BALANCE OF FORCES ENCODES NATIVE STRUCTURES?</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Anfinsen&#x02019;s Thermodynamic Hypothesis</h3><p id="P3" class="p p-first-last">A major milestone in protein science was the thermodynamic hypothesis of Christian Anfinsen and colleagues (<a href="#R3" rid="R3" class=" bibr popnode">3</a>, <a href="#R92" rid="R92" class=" bibr popnode">92</a>). From his now-famous experiments on ribonuclease, Anfinsen postulated that the native structure of a protein is the thermodynamically stable structure; it depends only on the amino acid sequence and on the conditions of solution, and not on the kinetic folding route. It became widely appreciated that the native structure does not depend on whether the protein was synthesized biologically on a ribosome or with the help of chaperone molecules, or if, instead, the protein was simply refolded as an isolated molecule in a test tube. [There are rare exceptions, however, such as insulin, &#x003b1;-lytic protease (<a href="#R203" rid="R203" class=" bibr popnode">203</a>), and the serpins (<a href="#R227" rid="R227" class=" bibr popnode">227</a>), in which the biologically active form is kinetically trapped.] Two powerful conclusions followed from Anfinsen&#x02019;s work. First, it enabled the large research enterprise of in vitro protein folding that has come to understand native structures by experiments inside test tubes rather than inside cells. Second, the Anfinsen principle implies a sort of division of labor: Evolution can act to change an amino acid sequence, but the folding equilibrium and kinetics of a given sequence are then matters of physical chemistry.</p></div><div id="S4" class="sec"><h3 id="S4title">One Dominant Driving Force or Many Small Ones?</h3><p id="P4" class="p p-first">Prior to the mid-1980s, the protein folding code was seen a sum of many different small interactions&#x02014;such as hydrogen bonds, ion pairs, van der Waals attractions, and water-mediated hydrophobic interactions. A key idea was that the primary sequence encoded secondary structures, which then encoded tertiary structures (<a href="#R4" rid="R4" class=" bibr popnode">4</a>). However, through statistical mechanical modeling, a different view emerged in the 1980s, namely, that there is a dominant component to the folding code, that it is the hydrophobic interaction, that the folding code is distributed both locally and nonlocally in the sequence, and that a protein&#x02019;s secondary structure is as much a consequence of the tertiary structure as a cause of it (<a href="#R48" rid="R48" class=" bibr popnode">48</a>, <a href="#R49" rid="R49" class=" bibr popnode">49</a>).</p><p id="P5">Because native proteins are only 5&#x02013;10 kcal/mol more stable than their denatured states, it is clear that no type of intermolecular force can be neglected in folding and structure prediction (<a href="#R238" rid="R238" class=" bibr popnode">238</a>). Although it remains challenging to separate in a clean and rigorous way some types of interactions from others, here are some of the main observations. Folding is not likely to be dominated by electrostatic interactions among charged side chains because most proteins have relatively few charged residues; they are concentrated in high-dielectric regions on the protein surface. Protein stabilities tend to be independent of pH (near neutral) and salt concentration, and charge mutations typically lead to small effects on structure and stability. Hydrogen-bonding interactions are important, because essentially all possible hydrogen-bonding interactions are generally satisfied in native structures. Hydrogen bonds among backbone amide and carbonyl groups are key components of all secondary structures, and studies of mutations in different solvents estimate their strengths to be around 1&#x02013;4 kcal/mol (<a href="#R21" rid="R21" class=" bibr popnode">21</a>, <a href="#R72" rid="R72" class=" bibr popnode">72</a>) or stronger (<a href="#R5" rid="R5" class=" bibr popnode">5</a>, <a href="#R46" rid="R46" class=" bibr popnode">46</a>). Similarly, tight packing in proteins implies that van der Waals interactions are important (<a href="#R28" rid="R28" class=" bibr popnode">28</a>).</p><p id="P6">However, the question of the folding code is whether there is a dominant factor that explains why any two proteins, for example, lysozyme and ribonuclease, have different native structures. This code must be written in the side chains, not in the backbone hydrogen bonding, because it is through the side chains that one protein differs from another. There is considerable evidence that hydrophobic interactions must play a major role in protein folding. (<em>a</em>) Proteins have hydrophobic cores, implying nonpolar amino acids are driven to be sequestered from water. (<em>b</em>) Model compound studies show 1&#x02013;2 kcal/mol for transferring a hydrophobic side chain from water into oil-like media (<a href="#R234" rid="R234" class=" bibr popnode">234</a>), and there are many of them. (<em>c</em>) Proteins are readily denatured in nonpolar solvents. (<em>d</em>) Sequences that are jumbled and retain only their correct hydrophobic and polar patterning fold to their expected native states (<a href="#R39" rid="R39" class=" bibr popnode">39</a>, <a href="#R98" rid="R98" class=" bibr popnode">98</a>, <a href="#R112" rid="R112" class=" bibr popnode">112</a>, <a href="#R118" rid="R118" class=" bibr popnode">118</a>), in the absence of efforts to design packing, charges, or hydrogen bonding. Hydrophobic and polar patterning also appears to be a key to encoding of amyloid-like fibril structures (<a href="#R236" rid="R236" class=" bibr popnode">236</a>).</p><p id="P7" class="p p-last">What stabilizes secondary structures? Before any protein structure was known, Linus Pauling and colleagues (<a href="#R180" rid="R180" class=" bibr popnode">180</a>, <a href="#R181" rid="R181" class=" bibr popnode">181</a>) inferred from hydrogen-bonding models that proteins might have &#x003b1;-helices. However, secondary structures are seldom stable on their own in solution. Although different amino acids have different energetic propensities to be in secondary structures (<a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R41" rid="R41" class=" bibr popnode">41</a>, <a href="#R55" rid="R55" class=" bibr popnode">55</a>, <a href="#R100" rid="R100" class=" bibr popnode">100</a>), there are also many &#x0201c;chameleon&#x0201d; sequences in natural proteins, which are peptide segments that can assume either helical or &#x003b2; conformations depending on their tertiary context (<a href="#R158" rid="R158" class=" bibr popnode">158</a>, <a href="#R162" rid="R162" class=" bibr popnode">162</a>). Studies of lattice models (<a href="#R25" rid="R25" class=" bibr popnode">25</a>, <a href="#R29" rid="R29" class=" bibr popnode">29</a>, <a href="#R51" rid="R51" class=" bibr popnode">51</a>) and tube models (<a href="#R11" rid="R11" class=" bibr popnode">11</a>, <a href="#R12" rid="R12" class=" bibr popnode">12</a>, <a href="#R159" rid="R159" class=" bibr popnode">159</a>) have shown that secondary structures in proteins are substantially stabilized by the chain compactness, an indirect consequence of the hydrophobic force to collapse (<a href="/pmc/articles/PMC2443096/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Like airport security lines, helical and sheet configurations are the only regular ways to pack a linear chain (of people or monomers) into a tight space.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2443096/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787308930176"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f1.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f1.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787308930176"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p>(<em>a</em>) Binary code. Experiments show that a primarily binary hydrophobic-polar code is sufficient to fold helix-bundle proteins (<a href="#R112" rid="R112" class=" bibr popnode">112</a>). Reprinted from Reference <a href="#R112" rid="R112" class=" bibr popnode">112</a> with permission from <em>AAAS</em>.</p><p>(<em>b</em>) Compactness stabilizes secondary structure, in proteins, from lattice models. (<em>c</em>) Experiments supporting panel <em>b</em>, showing that compactness correlates with secondary structure content in nonnative states of many different proteins (<a href="#R218" rid="R218" class=" bibr popnode">218</a>). Reprinted from Reference <a href="#R218" rid="R218" class=" bibr popnode">218</a> with permission.</p></div></div></div></div><div id="S5" class="sec sec-last"><h3 id="S5title">Designing New Proteins and Nonbiological Foldamers</h3><p id="P8" class="p p-first-last">Although our knowledge of the forces of folding remains incomplete, it has not hindered the emergence of successful practical protein design. Novel proteins are now designed as variants of existing proteins (<a href="#R43" rid="R43" class=" bibr popnode">43</a>, <a href="#R94" rid="R94" class=" bibr popnode">94</a>, <a href="#R99" rid="R99" class=" bibr popnode">99</a>, <a href="#R145" rid="R145" class=" bibr popnode">145</a>, <a href="#R173" rid="R173" class=" bibr popnode">173</a>, <a href="#R243" rid="R243" class=" bibr popnode">243</a>), or from broadened alphabets of nonnatural amino acids (<a href="#R226" rid="R226" class=" bibr popnode">226</a>), or de novo (<a href="#R129" rid="R129" class=" bibr popnode">129</a>) (<a href="/pmc/articles/PMC2443096/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Moreover, folding codes are used to design new polymeric materials called foldamers (<a href="#R76" rid="R76" class=" bibr popnode">76</a>, <a href="#R86" rid="R86" class=" bibr popnode">86</a>, <a href="#R120" rid="R120" class=" bibr popnode">120</a>). Folded helix bundles have now been designed using nonbiological backbones (<a href="#R134" rid="R134" class=" bibr popnode">134</a>). Foldamers are finding applications in biomedicine as antimicrobials (<a href="#R179" rid="R179" class=" bibr popnode">179</a>, <a href="#R185" rid="R185" class=" bibr popnode">185</a>), lung surfactant replacements (<a href="#R235" rid="R235" class=" bibr popnode">235</a>), cytomegalovirus inhibitors (<a href="#R62" rid="R62" class=" bibr popnode">62</a>), and siRNA delivery agents (<a href="#R217" rid="R217" class=" bibr popnode">217</a>). Hence, questions of deep principle are no longer bottlenecks to designing foldable polymers for practical applications and new materials.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2443096/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787297074896"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f2.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f2.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f2.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787297074896"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p>(<em>a</em>) A novel protein fold, called Top7, designed by Kuhlman et al. (<a href="#R129" rid="R129" class=" bibr popnode">129</a>). Designed molecule (<em>blue</em>) and the experimental structure determined subsequently (<em>red</em>). From Reference <a href="#R129" rid="R129" class=" bibr popnode">129</a>; reprinted with permission from <em>AAAS.</em> (<em>b</em>) Three-helix bundle foldamers have been made using nonbiological backbones (peptoids, i.e., N-substituted glycines).</p><p>(<em>c</em>) Their denaturation by alcohols indicates they have hydrophobic cores characteristic of a folded molecule (<a href="#R134" rid="R134" class=" bibr popnode">134</a>).</p></div></div></div></div></div><div id="S6" class="tsec sec"><h2 class="head no_bottom_margin" id="S6title">COMPUTATIONAL PROTEIN STRUCTURE PREDICTION IS INCREASINGLY SUCCESSFUL</h2><p id="P9" class="p p-first">A major goal of computational biology has been to predict a protein&#x02019;s three-dimensional native structure from its amino acid sequence. This could help to (<em>a</em>) accelerate drug discovery by replacing slow, expensive structural biology experiments with faster, cheaper computer simulations, and (<em>b</em>) annotate protein function from genome sequences (<a href="#R9" rid="R9" class=" bibr popnode">9</a>). With the rapid growth of experimentally determined structures available in the Protein Databank (PDB), protein structure prediction has become as much a problem of inference and machine learning as it is of protein physics.</p><p id="P10" class="p">Among the earliest uses of protein databases to infer protein structures were secondary structure prediction algorithms (<a href="#R33" rid="R33" class=" bibr popnode">33</a>, <a href="#R34" rid="R34" class=" bibr popnode">34</a>, <a href="#R190" rid="R190" class=" bibr popnode">190</a>). In the mid-1980s, several groups began using the methods of computational physics&#x02014;atomic force fields plus Monte Carlo sampling&#x02014;to compute the structures of the Metenkephalin, a five-residue peptide (<a href="#R95" rid="R95" class=" bibr popnode">95</a>, <a href="#R141" rid="R141" class=" bibr popnode">141</a>). The early 1990s saw significant strides in using databases and homology detection algorithms to assemble structures from homologous sequences (<a href="#R192" rid="R192" class=" bibr popnode">192</a>) and to recognize folds by threading unknown sequences onto three-dimensional structures from a database (<a href="#R111" rid="R111" class=" bibr popnode">111</a>). A key advance was the exploitation of evolutionary relationships among sequences through the development of robust sequence alignment methods (<a href="#R32" rid="R32" class=" bibr popnode">32</a>, <a href="#R64" rid="R64" class=" bibr popnode">64</a>, <a href="#R224" rid="R224" class=" bibr popnode">224</a>).</p><div id="S7" class="sec sec-last"><h3 id="S7title">CASP: A Community-Wide Experiment</h3><p id="P11" class="p p-first">In 1994, John Moult invented CASP (Critical Assessment of Techniques for Protein Structure Prediction) (<a href="#R165" rid="R165" class=" bibr popnode">165</a>), a biennial, community-wide blind test to predict the unknown structures of proteins. Organizers identify proteins likely to be solved or whose structures have not yet been released, and predictors have roughly 3&#x02013;5 weeks to predict their native structures. CASP has grown from 35 predictor groups and 24 target sequences in CASP1 in 1994 to over 200 groups and 100+ targets in CASP7 in 2006.</p><p id="P12">Over the seven CASPs, two trends are clear (<a href="#R164" rid="R164" class=" bibr popnode">164</a>, <a href="#R219" rid="R219" class=" bibr popnode">219</a>). First, although much remains to be done, there has been substantial improvement in protein structure prediction. Web servers and software packages often predict the native structure of small, single-domain proteins to within about 2&#x02013;6 &#x000c5; of their experimental structures (<a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R17" rid="R17" class=" bibr popnode">17</a>, <a href="#R242" rid="R242" class=" bibr popnode">242</a>). In addition, fast-homology methods are computing approximate folds for whole genomes (<a href="#R182" rid="R182" class=" bibr popnode">182</a>, <a href="#R214" rid="R214" class=" bibr popnode">214</a>). <a href="/pmc/articles/PMC2443096/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows a quantitative assessment of performance at the first five CASP meetings. The most significant gains have occurred in the alignments of targets to homologs, the detection of evolutionarily distant homologs, and the generation of reasonable models for difficult targets that do not have templates (new folds). Since CASP5, predictions have also benefited from the use of metaservers, which solicit and establish consensus among predictions from multiple algorithms. Second, while most methods rely on both physics and bioinformatics, the most successful methods currently draw heavily from knowledge contained in native structural databases. Bioinformatics methods have benefited from the growth in size of the PDB (<a href="#R9" rid="R9" class=" bibr popnode">9</a>, <a href="#R219" rid="R219" class=" bibr popnode">219</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2443096/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787297204752"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f3.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f3.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f3.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787297204752"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Progress in protein structure prediction in CASP1&#x02013;5 (<a href="#R219" rid="R219" class=" bibr popnode">219</a>). The y-axis contains the GDT TS score, the percentage of model residues that can be superimposed on the true native structure, averaged over four resolutions from 1 to 8 &#x000c5; (100% is perfect). The x-axis is the ranked target difficulty, measured by sequence and structural similarities to proteins in the PDB at the time of the respective CASP. This shows that protein structure prediction on easy targets is quite good and is improving for targets of intermediate difficulty. Reprinted from Reference <a href="#R219" rid="R219" class=" bibr popnode">219</a> with permission.</p></div></div></div><p id="P13" class="p p-last">The following challenges remain (<a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R164" rid="R164" class=" bibr popnode">164</a>, <a href="#R219" rid="R219" class=" bibr popnode">219</a>): (<em>a</em>) to refine homology models beyond those of the best template structures; (<em>b</em>) to reduce errors to routinely better than 3 &#x000c5;, particularly for proteins that are large, have significant &#x003b2; content, are new folds, or have low homology; (<em>c</em>) to handle large multidomain or domain-swapped proteins; (<em>d</em>) to address membrane proteins; and (<em>e</em>) to predict protein-protein interactions. Structural genomics is likely to help here (<a href="#R87" rid="R87" class=" bibr popnode">87</a>, <a href="#R222" rid="R222" class=" bibr popnode">222</a>). In any case, the current successes in computer-based predictions of native protein structures are far beyond what was expected thirty years ago, when structure prediction seemed impossible.</p></div></div><div id="S8" class="tsec sec"><h2 class="head no_bottom_margin" id="S8title">ARE THERE MECHANISMS OF PROTEIN FOLDING?</h2><p id="P14" class="p p-first">In 1968, Cyrus Levinthal first noted the puzzle that even though they have vast conformational spaces, proteins can search and converge quickly to native states, sometimes in microseconds. How do proteins find their native states so quickly? It was postulated that if we understood the physical mechanism of protein folding, it could lead to fast computer algorithms to predict native structures from their amino acid sequences. In its description of the 125 most important unsolved problems in science, <em>Science</em> magazine framed the problem this way: &#x0201c;Can we predict how proteins will fold? Out of a near infinitude of possible ways to fold, a protein picks one in just tens of microseconds. The same task takes 30 years of computer time&#x0201d; (<a href="#R1" rid="R1" class=" bibr popnode">1</a>).</p><p id="P15">The following questions of principle have driven the field: How can all the denatured molecules in a beaker find the same native structure, starting from different conformations? What conformations are not searched? Is folding hierarchical (<a href="#R10" rid="R10" class=" bibr popnode">10</a>, <a href="#R119" rid="R119" class=" bibr popnode">119</a>)? Which comes first: secondary or tertiary structure (<a href="#R80" rid="R80" class=" bibr popnode">80</a>, <a href="#R239" rid="R239" class=" bibr popnode">239</a>)? Does the protein collapse to compact structures before structure formation, or before the rate-limiting step (RLS), or are they concurrent (<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R89" rid="R89" class=" bibr popnode">89</a>, <a href="#R101" rid="R101" class=" bibr popnode">101</a>, <a href="#R195" rid="R195" class=" bibr popnode">195</a>, <a href="#R205" rid="R205" class=" bibr popnode">205</a>, <a href="#R213" rid="R213" class=" bibr popnode">213</a>)? Are there folding nuclei (<a href="#R58" rid="R58" class=" bibr popnode">58</a>, <a href="#R152" rid="R152" class=" bibr popnode">152</a>)?</p><p id="P16" class="p">Several models have emerged. In the diffusion-collision model, microdomain structures form first and then diffuse and collide to form larger structures (<a href="#R115" rid="R115" class=" bibr popnode">115</a>, <a href="#R116" rid="R116" class=" bibr popnode">116</a>). The nucleation-condensation mechanism (<a href="#R70" rid="R70" class=" bibr popnode">70</a>) proposes that a diffuse transition state ensemble (TSE) with some secondary structure nucleates tertiary contacts. Some proteins, such as helical bundles, appear to follow a hierarchical diffusion-collision model (<a href="#R155" rid="R155" class=" bibr popnode">155</a>, <a href="#R169" rid="R169" class=" bibr popnode">169</a>) in which secondary structure forms and assembles in a hierarchical order. In hierarchic condensation (<a href="#R139" rid="R139" class=" bibr popnode">139</a>), the chain searches for compact, contiguous structured units, which are then assembled into the folded state. Or, proteins may fold via the stepwise assembly of structural subunits called foldons (<a href="#R22" rid="R22" class=" bibr popnode">22</a>, <a href="#R126" rid="R126" class=" bibr popnode">126</a>), or they may search for topomers, which are largely unfolded states that have native-like topologies (<a href="#R45" rid="R45" class=" bibr popnode">45</a>, <a href="#R150" rid="R150" class=" bibr popnode">150</a>). These models are not mutually exclusive.</p><div id="S9" class="sec"><h3 id="S9title">There Have Been Big Advances in Experimental and Theoretical Methods</h3><p id="P17" class="p p-first">The search for folding mechanisms has driven major advances in experimental protein science. These include fast laser temperature-jump methods (<a href="#R22" rid="R22" class=" bibr popnode">22</a>); mutational methods that give quantities called &#x003c6;-values (<a href="#R71" rid="R71" class=" bibr popnode">71</a>, <a href="#R84" rid="R84" class=" bibr popnode">84</a>, <a href="#R106" rid="R106" class=" bibr popnode">106</a>) [now also used for ion-channel kinetics and other rate processes (<a href="#R42" rid="R42" class=" bibr popnode">42</a>)] or &#x003c8;-values (<a href="#R204" rid="R204" class=" bibr popnode">204</a>), which can identify those residues most important for folding speed; hydrogen exchange methods that give monomer-level information about folding events (<a href="#R125" rid="R125" class=" bibr popnode">125</a>, <a href="#R149" rid="R149" class=" bibr popnode">149</a>); and the extensive exploration of protein model systems, including cytochrome <em>c</em>, CI2, barnase, apomyoglobin, the src, &#x003b1;-spectrin, and fyn SH3 domains, proteins L and G, WW domains, trpzip, and trp cage (<a href="#R154" rid="R154" class=" bibr popnode">154</a>). In addition, peptide model experimental test systems provide insights into the fast early-folding events (<a href="#R14" rid="R14" class=" bibr popnode">14</a>, <a href="#R109" rid="R109" class=" bibr popnode">109</a>, <a href="#R124" rid="R124" class=" bibr popnode">124</a>). Furthermore, single-molecule methods are beginning to explore the conformational heterogeneity of folding (<a href="#R23" rid="R23" class=" bibr popnode">23</a>, <a href="#R133" rid="R133" class=" bibr popnode">133</a>, <a href="#R166" rid="R166" class=" bibr popnode">166</a>, <a href="#R194" rid="R194" class=" bibr popnode">194</a>).</p><p id="P18" class="p p-last">There have been corresponding advances in theory and computation. Computer-based molecular minimization methods were first applied to protein structures in the 1960s (<a href="#R18" rid="R18" class=" bibr popnode">18</a>, <a href="#R79" rid="R79" class=" bibr popnode">79</a>, <a href="#R171" rid="R171" class=" bibr popnode">171</a>), followed by molecular dynamics (<a href="#R140" rid="R140" class=" bibr popnode">140</a>, <a href="#R155" rid="R155" class=" bibr popnode">155</a>), improved force fields (<a href="#R40" rid="R40" class=" bibr popnode">40</a>) (reviewed in Reference <a href="#R114" rid="R114" class=" bibr popnode">114</a>), weighted sampling and multi-temperature methods (<a href="#R130" rid="R130" class=" bibr popnode">130</a>, <a href="#R210" rid="R210" class=" bibr popnode">210</a>), highly parallelized codes for supercomputers (<a href="#R2" rid="R2" class=" bibr popnode">2</a>, <a href="#R57" rid="R57" class=" bibr popnode">57</a>), and distributed grid computing methods such as Folding@home (<a href="#R198" rid="R198" class=" bibr popnode">198</a>, <a href="#R241" rid="R241" class=" bibr popnode">241</a>). Models having less atomic detail also emerged to address questions more global and less detailed in nature about protein conformational spaces: (<em>a</em>) The Go model (<a href="#R82" rid="R82" class=" bibr popnode">82</a>), which was intended to see if a computer could find the native structure if native guiding constraints were imposed, is now widely used to study folding kinetics of proteins having known native structures (<a href="#R37" rid="R37" class=" bibr popnode">37</a>, <a href="#R38" rid="R38" class=" bibr popnode">38</a>, <a href="#R113" rid="R113" class=" bibr popnode">113</a>, <a href="#R197" rid="R197" class=" bibr popnode">197</a>, <a href="#R225" rid="R225" class=" bibr popnode">225</a>). (<em>b</em>) More physical models, typically based on polymer-like lattices, are used to study the static and dynamic properties of conformational spaces (<a href="#R19" rid="R19" class=" bibr popnode">19</a>, <a href="#R50" rid="R50" class=" bibr popnode">50</a>). (<em>c</em>) Master-equation approaches can explore dynamics in heterogeneous systems (<a href="#R26" rid="R26" class=" bibr popnode">26</a>, <a href="#R36" rid="R36" class=" bibr popnode">36</a>, <a href="#R77" rid="R77" class=" bibr popnode">77</a>, <a href="#R138" rid="R138" class=" bibr popnode">138</a>, <a href="#R161" rid="R161" class=" bibr popnode">161</a>, <a href="#R176" rid="R176" class=" bibr popnode">176</a>, <a href="#R231" rid="R231" class=" bibr popnode">231</a>, <a href="#R232" rid="R232" class=" bibr popnode">232</a>). Below, we describe some of what has been learned from these studies.</p></div><div id="S10" class="sec"><h3 id="S10title">The PSB Plot: Folding Speed Correlates with the Localness of Contacts in the Native Structure</h3><p id="P19" class="p p-first-last">One of the few universal features of protein folding kinetics was first observed by Plaxco, Simons, and Baker (PSB), namely, that the folding speed of a protein is correlated with a topological property of its native structure (<a href="#R88" rid="R88" class=" bibr popnode">88</a>, <a href="#R184" rid="R184" class=" bibr popnode">184</a>). As shown in <a href="/pmc/articles/PMC2443096/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>, Plaxco et al. found that the folding rates of two-state proteins&#x02014;now known to vary more than 8 orders of magnitude&#x02014;correlate with the average degree to which native contacts are local within the chain sequence: Fast-folders usually have mostly local structure, such as helices and tight turns. Slow-folders usually have more nonlocal structure, such as &#x003b2;-sheets (<a href="#R184" rid="R184" class=" bibr popnode">184</a>), although there are exceptions (<a href="#R237" rid="R237" class=" bibr popnode">237</a>). Folding rates have been subsequently found to correlate well with other native topological parameters such as the protein&#x02019;s effective chain length (chain length minus the number of amino acids in helices) (<a href="#R107" rid="R107" class=" bibr popnode">107</a>), secondary structure length (<a href="#R104" rid="R104" class=" bibr popnode">104</a>), sequence-distant contacts per residue (<a href="#R90" rid="R90" class=" bibr popnode">90</a>), the fraction of contacts that are sequence distant (<a href="#R163" rid="R163" class=" bibr popnode">163</a>), the total contact distance (<a href="#R245" rid="R245" class=" bibr popnode">245</a>), and intrinsic propensities, for example, of &#x003b1;-helices (<a href="#R131" rid="R131" class=" bibr popnode">131</a>). And, there are now also methods that predict the folding rate from the sequence (<a href="#R91" rid="R91" class=" bibr popnode">91</a>, <a href="#R186" rid="R186" class=" bibr popnode">186</a>). It follows that a protein typically forms smaller loops and turns faster than it forms larger loops and turns, consistent with the so-called zipping and assembly (ZA) mechanism, described below, which postulates that search speed is governed by the effective loop sizes [the effective contact order (ECO) (<a href="#R53" rid="R53" class=" bibr popnode">53</a>, <a href="#R73" rid="R73" class=" bibr popnode">73</a>)] that the chain must search at any step.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2443096/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787294911808"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f4.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f4.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f4.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787294911808"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p>Folding rate versus relative contact order (a measure of localness of contacts in the native structure) for the 48 two-state proteins given in Reference <a href="#R91" rid="R91" class=" bibr popnode">91</a>, showing that proteins with the most local contacts fold faster than proteins with more nonlocal contacts.</p></div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Proteins Fold on Funnel-Shaped Energy Landscapes</h3><p id="P20" class="p p-first">Why has folding been regarded as so challenging? The issue is the astronomical number of conformations a protein must search to find its native state. Models arose in the 1980s to study the nature of the conformational space (<a href="#R19" rid="R19" class=" bibr popnode">19</a>, <a href="#R47" rid="R47" class=" bibr popnode">47</a>), i.e., the shape of the energy landscape, which is the mathematical function F(&#x003c6;, &#x003d5;, X) that describes the intramolecular-plus-solvation free energy of a given protein as a function of the microscopic degrees of freedom. A central goal has been to quantify the statistical mechanical partition function, a key component of which is the density of states (DOS), i.e., the number of conformations at each energy level. In simple cases, the logarithm of the DOS is the conformational entropy. Such entropies have not been determinable through all-atom modeling, because that would require astronomical amounts of computational sampling (although replica-exchange methods can now do this for very small peptides). Hence understanding a protein&#x02019;s DOS and its entropies has required simplified models, such as mean-field polymer and lattice treatments (<a href="#R51" rid="R51" class=" bibr popnode">51</a>), spin-glass theories (<a href="#R19" rid="R19" class=" bibr popnode">19</a>, <a href="#R47" rid="R47" class=" bibr popnode">47</a>), or exact enumerations in minimalist models (<a href="#R132" rid="R132" class=" bibr popnode">132</a>).</p><p id="P21" class="p">A key conclusion is that proteins have funnel-shaped energy landscapes, i.e., many high-energy states and few low-energy states (<a href="#R19" rid="R19" class=" bibr popnode">19</a>, <a href="#R49" rid="R49" class=" bibr popnode">49</a>, <a href="#R50" rid="R50" class=" bibr popnode">50</a>, <a href="#R52" rid="R52" class=" bibr popnode">52</a>, <a href="#R138" rid="R138" class=" bibr popnode">138</a>). What do we learn from the funnel idea? Funnels have both qualitative and quantitative uses. First, cartoonizations of funnels (<a href="/pmc/articles/PMC2443096/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>) provide a useful shorthand language for communicating the statistical mechanical properties and folding kinetics of proteins. <a href="/pmc/articles/PMC2443096/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a> illustrates fast folding (simple funnel), kinetic trapping (moats or wells), and slow random searching (golf course). These pictures show a key distinction between protein folding and simple classical chemical reactions. A simple chemical reaction proceeds from its reactant A to its product B, through a pathway, i.e., a sequence of individual structures. A protein cannot do this because its reactant, the denatured state, is not a single microscopic structure. Folding is a transition from disorder to order, not from one structure to another. Simple one-dimensional reaction path diagrams do not capture this tremendous reduction in conformational degeneracy.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2443096/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787289372000"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f5.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f5.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f5.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787289372000"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p>One type of energy landscape cartoon: the free energy F(&#x003c6;, &#x003d5;, &#x003ba;) of the bond degrees of freedom. These pictures give a sort of simplified schematic diagram, useful for illustrating a protein&#x02019;s partition function and density of states. (<em>a</em>) A smooth energy landscape for a fast folder, (<em>b</em>) a rugged energy landscape with kinetic traps, (<em>c</em>) a golf course energy landscape in which folding is dominated by diffusional conformational search, and (<em>d</em>) a moat landscape, where folding must pass through an obligatory intermediate.</p></div></div></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">A funnel describes a protein&#x02019;s conformational heterogeneity </h4><p id="P22" class="p p-first-last">Conformational heterogeneity has been found in the few experiments that have been designed to look for it (<a href="#R16" rid="R16" class=" bibr popnode">16</a>, <a href="#R143" rid="R143" class=" bibr popnode">143</a>, <a href="#R157" rid="R157" class=" bibr popnode">157</a>, <a href="#R206" rid="R206" class=" bibr popnode">206</a>, <a href="#R209" rid="R209" class=" bibr popnode">209</a>, <a href="#R244" rid="R244" class=" bibr popnode">244</a>). For example, using time-resolved FRET with four different intramolecular distances, Sridevi et al. (<a href="#R206" rid="R206" class=" bibr popnode">206</a>) found in Barstar that (<em>a</em>) that the chain entropy increases as structures become less stable, (<em>b</em>) that there are multiple folding routes, and (<em>c</em>) that different routes dominate under different folding conditions. Moreover, changing the denaturant can change the dominant pathway, implying heterogeneous kinetics (<a href="#R143" rid="R143" class=" bibr popnode">143</a>). <a href="/pmc/articles/PMC2443096/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6</span></a> shows the funnel landscape that has been determined by extensive mutational analysis of the seven ankyrin sequence repeats of the Notch ankyrin repeat domain (<a href="#R16" rid="R16" class=" bibr popnode">16</a>, <a href="#R157" rid="R157" class=" bibr popnode">157</a>, <a href="#R209" rid="R209" class=" bibr popnode">209</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC2443096/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787348811120"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f6.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f6.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f6.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787348811120"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a7--><div class="caption"><p>The experimentally determined energy landscape of the seven ankyrin repeats of the Notch receptor (<a href="#R16" rid="R16" class=" bibr popnode">16</a>, <a href="#R157" rid="R157" class=" bibr popnode">157</a>, <a href="#R209" rid="R209" class=" bibr popnode">209</a>). The energy landscape is constructed by measuring the stabilities of folded fragments for a series of overlapping modular repeats. Each horizontal tier presents the partially folded fragments with the same number of repeats. Reprinted from Reference <a href="#R157" rid="R157" class=" bibr popnode">157</a> with permission.</p></div></div></div></div><div id="S13" class="sec"><p></p><h4 id="S13title" class="inline">A funnel describes a protein&#x02019;s chain entropy </h4><p id="P23" class="p p-first-last">The funnel idea first arose to explain denaturation, the balance between the chain entropy and the forces of folding (<a href="#R48" rid="R48" class=" bibr popnode">48</a>). Proteins denature at high temperatures because there are many states of high energy and fewer states of low energy, that is, the landscape is funneled. For cold unfolding, the shape of the funnel changes with temperature because of free-energy changes of the aqueous solvent. When you can accurately compute a protein&#x02019;s DOS, you can predict the protein&#x02019;s free energy of folding and its denaturation and cooperativity properties. <a href="/pmc/articles/PMC2443096/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figure 7</span></a> shows an example in which the DOS (set onto its side to illustrate the funnel) was found by extensive lattice enumeration for F13W*, a three-helix bundle, with predictions compared to experiments (<a href="#R146" rid="R146" class=" bibr popnode">146</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC2443096/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787296183520"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f7.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f7.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f7.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787296183520"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7</a></div><!--caption a7--><div class="caption"><p>(<em>Left</em>) The density of states (DOS) cartoonized as an energy landscape for the three-helix bundle protein F13W*: DOS (x-axis) versus the energy (y-axis). (<em>Right</em>) Denaturation predictions versus experiments (<a href="#R146" rid="R146" class=" bibr popnode">146</a>). The peak free energy (here, where the DOS is minimum), typically taken to be the transition state, is energetically very close to native.</p></div></div></div></div><div id="S14" class="sec"><p></p><h4 id="S14title" class="inline">Funnels provide a microscopic framework for folding kinetics </h4><p id="P24" class="p p-first">Folding kinetics is traditionally described by simple mass action models, such as D &#x02192;I &#x02192;N (on-path intermediate I between the denatured state D and native state N) or X &#x02192;D &#x02192;N (off-path intermediate X), where the symbol I or X represents macrostates that are invoked for the purpose of curve-fitting experimental kinetics data. In contrast, funnel models or master-equation models aim to explain the kinetics in terms of underlying physical forces. They aim to predict the microstate composition of those macrostates, for example. The states in master-equation models differ from those in mass-action models insofar as the former are more numerous, more microscopic, are defined by structural or energetic criteria, and are arranged kinetically to reflect the underlying funnel-like organization of the dynamical flows.</p><p id="P25">For example, <a href="/pmc/articles/PMC2443096/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Figure 8<em>b</em></span></a> shows a master-equation model for the folding of SH3, illustrating the apparently paradoxical result that folding can be serial and parallel at the same time. The protein has multiple routes available. However, one of the dominant series pathways is U &#x02192;B &#x02192;BD &#x02192;BDE &#x02192;BCDE &#x02192;N. BD is the TSE because it is the dynamically least populated state. B precedes BD diagrammatically in series in this pathway. Yet, the probability bucket labeled B does not first fill up and then empty into BD; rather the levels in both buckets, B and BD, rise and fall together and hence are dynamically in parallel. Such series and parallel steps are also seen in computer simulations of CI2 (<a href="#R189" rid="R189" class=" bibr popnode">189</a>), for example. Sometimes a chain contact A forms before another contact B in nearly all the simulation trajectories (series-like). But another contact C may form before B in some trajectories and after B in others (parallel-like). Some folding is sequential, as in Fyn SH3 (<a href="#R123" rid="R123" class=" bibr popnode">123</a>), cytochrome (<a href="#R66" rid="R66" class=" bibr popnode">66</a>), T4 lysozyme (<a href="#R24" rid="R24" class=" bibr popnode">24</a>), and Im7 (<a href="#R74" rid="R74" class=" bibr popnode">74</a>), and some folding is parallel, as in cytochrome C (<a href="#R83" rid="R83" class=" bibr popnode">83</a>) and HEW lyzosyme (<a href="#R151" rid="R151" class=" bibr popnode">151</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC2443096/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787295285136"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f8.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f8.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f8.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787295285136"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8</a></div><!--caption a7--><div class="caption"><p>(<em>a</em>) A simple single-pathway system. R, reactant; TS, transition state; P, product. (<em>b</em>) Pathway diagram of SH3 folding from a master-equation model. The native protein has five contact clusters: A = RT loop; B = &#x003b2;<sub>2</sub>&#x003b2;<sub>3</sub>; C = &#x003b2;<sub>3</sub>&#x003b2;<sub>4</sub>; D = RT loop-&#x003b2;<sub>4</sub>; and E = &#x003b2;<sub>1</sub>&#x003b2;<sub>5</sub>. Combined letters, such as BD, mean that multiple contact clusters have formed. Funneling occurs toward the right, because the symbols on the left indicate large ensembles, whereas the symbols on the right are smaller ensembles. The numbers indicate free energies relative to the denatured state. The arrows between the states are colored to indicate transition times between states. The slowest steps are in red; the fastest steps in green. BD is the transition state ensemble because it is the highest free energy along the dominant route. While B and BD would seem to be obligatorily in series, the time evolutions of these states show that they actually rise and fall in parallel (<a href="#R232" rid="R232" class=" bibr popnode">232</a>).</p></div></div></div><p id="P26" class="p p-last">Or, consider the traditional idea that a reaction&#x02019;s RLS coincides with the point of the highest free energy, &#x00394;G<sup>&#x02021;</sup>, along the reaction coordinate. As a matter of principle, the RLS need not necessarily coincide with the free-energy barrier. There is evidence that they may not coincide for some protein folding processes (<a href="#R13" rid="R13" class=" bibr popnode">13</a>, <a href="#R15" rid="R15" class=" bibr popnode">15</a>, <a href="#R27" rid="R27" class=" bibr popnode">27</a>). What&#x02019;s the distinction? To find the RLS, you need a dynamical model. You would find the eigenvector corresponding to the slowest eigenvalue (in two-state kinetics). Microscopic master-equation modeling typically finds that this eigenvector only identifies the process U &#x02192;N, with no finer pinpointing of any special structures along the way. In contrast, &#x00394;G<sup>&#x02021;</sup> is a thermodynamic quantity&#x02014;the maximum free energy along the fastest route, which usually does correspond to some specific ensemble of structures. Below, we describe why these matters of principle are important.</p></div><div id="S15" class="sec"><p></p><h4 id="S15title" class="inline">How do we convert folding experiments into insights about molecular behavior? </h4><p id="P27" class="p p-first-last">To interpret data, we must use models. &#x003c6;-value experiments aim to identify RLSs in folding. But how we understand the molecular events causing a given &#x003c6;-value depends on whether we interpret it by funnels or pathways. A &#x003c6;-value measures how a folding rate changes when a protein is mutated (<a href="#R42" rid="R42" class=" bibr popnode">42</a>, <a href="#R67" rid="R67" class=" bibr popnode">67</a>, <a href="#R71" rid="R71" class=" bibr popnode">71</a>, <a href="#R105" rid="R105" class=" bibr popnode">105</a>, <a href="#R144" rid="R144" class=" bibr popnode">144</a>, <a href="#R153" rid="R153" class=" bibr popnode">153</a>, <a href="#R154" rid="R154" class=" bibr popnode">154</a>, <a href="#R176" rid="R176" class=" bibr popnode">176</a>, <a href="#R178" rid="R178" class=" bibr popnode">178</a>, <a href="#R193" rid="R193" class=" bibr popnode">193</a>) (see Reference <a href="#R231" rid="R231" class=" bibr popnode">231</a> and references <a href="#R1" rid="R1" class=" bibr popnode">1</a>&#x02013;<a href="#R24" rid="R24" class=" bibr popnode">24</a> therein). &#x003c6; equals the change in the logarithm of the folding rate caused by the mutation, divided by the change in the logarithm of the folding equilibrium constant. If we then seek a structural interpretation of &#x003c6;, we need a model. Using the Bronsted-Hammond pathway model of chemical reactions, &#x003c6; is often assumed to indicate the position of the TSE along the folding reaction coordinate: &#x003c6; = 0 means the mutation site is denatured in the TSE; &#x003c6; = 1 means the mutation site is native in the TSE. In this pathway view, &#x003c6; can never lie outside the range from 0 to 1; in the funnel view, &#x003c6; is not physically restricted to this range. For example, &#x003c6; &#x0003c; 0 or &#x003c6; &#x0003e; 1 has been predicted for mutations that stabilize a helix but that destabilize the bundle&#x02019;s tertiary structure (<a href="#R231" rid="R231" class=" bibr popnode">231</a>). Unfortunately, experiments are not yet definitive. While some &#x003c6;-values are indeed observed to be negative or greater than 1 (<a href="#R44" rid="R44" class=" bibr popnode">44</a>, <a href="#R85" rid="R85" class=" bibr popnode">85</a>, <a href="#R176" rid="R176" class=" bibr popnode">176</a>, <a href="#R193" rid="R193" class=" bibr popnode">193</a>), those values might be experimental artifacts (<a href="#R193" rid="R193" class=" bibr popnode">193</a>). Other challenges in interpreting &#x003c6; have also been noted (<a href="#R65" rid="R65" class=" bibr popnode">65</a>, <a href="#R188" rid="R188" class=" bibr popnode">188</a>). To resolve the ambiguities in interpreting &#x003c6;, we need to deepen our understanding beyond the single-reaction-coordinate idea.</p></div><div id="S16" class="sec"><p></p><h4 id="S16title" class="inline">How do we convert computer simulations into insights about molecular behavior? </h4><p id="P28" class="p p-first-last">Similarly, insights about folding events are often sought from computer modeling. It is much easier to calculate structural or energetic quantities than kinetic quantities. For example, some modeling efforts compute &#x003c6;-values by assuming some particular structure for the TSE (<a href="#R78" rid="R78" class=" bibr popnode">78</a>, <a href="#R233" rid="R233" class=" bibr popnode">233</a>) or some particular reaction coordinate, such as the RMSD to native structure, radius of gyration, number of hydrogen bonds, or number of native contacts (<a href="#R196" rid="R196" class=" bibr popnode">196</a>). Alternatively, a quantity called pfold (<a href="#R56" rid="R56" class=" bibr popnode">56</a>), which defines a separatrix (a sort of continental divide between folded and unfolded states), is sometimes computed. Although pfold predicts well the RLSs for simple landscapes (<a href="#R147" rid="R147" class=" bibr popnode">147</a>), it can give less insight into protein landscapes having multiple barriers or other complexity (<a href="#R30" rid="R30" class=" bibr popnode">30</a>). To go beyond classical assumptions, there has been an extensive and growing effort to use master-equation approaches (<a href="#R13" rid="R13" class=" bibr popnode">13</a>, <a href="#R26" rid="R26" class=" bibr popnode">26</a>, <a href="#R31" rid="R31" class=" bibr popnode">31</a>, <a href="#R36" rid="R36" class=" bibr popnode">36</a>, <a href="#R60" rid="R60" class=" bibr popnode">60</a>, <a href="#R61" rid="R61" class=" bibr popnode">61</a>, <a href="#R77" rid="R77" class=" bibr popnode">77</a>, <a href="#R110" rid="R110" class=" bibr popnode">110</a>, <a href="#R161" rid="R161" class=" bibr popnode">161</a>, <a href="#R172" rid="R172" class=" bibr popnode">172</a>, <a href="#R176" rid="R176" class=" bibr popnode">176</a>, <a href="#R201" rid="R201" class=" bibr popnode">201</a>, <a href="#R202" rid="R202" class=" bibr popnode">202</a>, <a href="#R208" rid="R208" class=" bibr popnode">208</a>, <a href="#R211" rid="R211" class=" bibr popnode">211</a>, <a href="#R212" rid="R212" class=" bibr popnode">212</a>, <a href="#R231" rid="R231" class=" bibr popnode">231</a>, <a href="#R232" rid="R232" class=" bibr popnode">232</a>) to explore underlying assumptions about reaction coordinates, pathways, transition states, and RLSs.</p></div><div id="S17" class="sec sec-last"><p></p><h4 id="S17title" class="inline">Funnel models can explain some non-canonical behaviors in ultrafast folding </h4><p id="P29" class="p p-first">More than a dozen proteins fold in microseconds (<a href="#R128" rid="R128" class=" bibr popnode">128</a>). Some fold in hundreds of nanoseconds (<a href="#R127" rid="R127" class=" bibr popnode">127</a>, <a href="#R237" rid="R237" class=" bibr popnode">237</a>). Is there a state of protein folding that is so fast that there is no free-energy barrier at all (<a href="#R156" rid="R156" class=" bibr popnode">156</a>)? This has sometimes been called downhill folding (<a href="#R93" rid="R93" class=" bibr popnode">93</a>, <a href="#R122" rid="R122" class=" bibr popnode">122</a>, <a href="#R128" rid="R128" class=" bibr popnode">128</a>, <a href="#R187" rid="R187" class=" bibr popnode">187</a>). There is currently an intensive search for downhill folders&#x02014;and much controversy about whether or not such folding has yet been observed, mainly in BBL, a 40-residue helical protein. That controversy hinges on questions of experimental analysis (<a href="#R68" rid="R68" class=" bibr popnode">68</a>, <a href="#R69" rid="R69" class=" bibr popnode">69</a>, <a href="#R75" rid="R75" class=" bibr popnode">75</a>, <a href="#R103" rid="R103" class=" bibr popnode">103</a>, <a href="#R168" rid="R168" class=" bibr popnode">168</a>, <a href="#R170" rid="R170" class=" bibr popnode">170</a>, <a href="#R191" rid="R191" class=" bibr popnode">191</a>): establishing proper baselines and ionization states to find denaturation temperatures and to determine whether the equilibrium is two-state, for example, which would imply a barrier between D and N.</p><p id="P30" class="p p-last">Remarkably, all known ultrafast-folders have anti-Arrhenius thermal kinetics. That is, heating those proteins at high temperatures slows down folding, the opposite of what is expected from traditional activation barriers. Here too, any molecular interpretation requires a model, and the common expectation is based on the classical Arrhenius/Eyring pathway model. Is the Arrhenius model sufficient for funnels involving many fast processes? Ultrafast folding kinetics has recently been explored in various models (<a href="#R59" rid="R59" class=" bibr popnode">59</a>, <a href="#R77" rid="R77" class=" bibr popnode">77</a>, <a href="#R122" rid="R122" class=" bibr popnode">122</a>, <a href="#R148" rid="R148" class=" bibr popnode">148</a>, <a href="#R167" rid="R167" class=" bibr popnode">167</a>). One funnel model (<a href="#R77" rid="R77" class=" bibr popnode">77</a>) explains that the reason why increasing the temperature leads to slower folding is because of thermal unfolding of the denatured chain, leading to a larger conformational space that must be searched for the chain to find route to native downhill. It predicts that the ultimate speed limit to protein folding, at temperatures that will disappear all other barriers, is the conformational search through the denatured basin. Near the speed limit of protein folding, the heterogeneity and searching that are intrinsic to funnels can be an important component of the folding physics. That model also explains that helical proteins fold faster than &#x003b2;-sheets, on average, because helices have more parallel microscopic folding routes (because a helix can nucleate at many different points along the chain).</p></div></div><div id="S18" class="sec sec-last"><h3 id="S18title">The Zipping and Assembly Hypothesis for the Folding Routes</h3><p id="P31" class="p p-first">Protein folding is a stochastic process: One protein molecule in a beaker follows a different microscopic trajectory than another molecule because of thermal fluctuations. Hence, protein folding is often studied using Monte Carlo or molecular dynamics sampling. However, computations seeking the native state using purely physical models are prohibitively expensive, because this is a challenging needle-in-a-haystack global optimization problem (<a href="#R96" rid="R96" class=" bibr popnode">96</a>, <a href="#R132" rid="R132" class=" bibr popnode">132</a>, <a href="#R216" rid="R216" class=" bibr popnode">216</a>). Since the beginnings of experimental folding kinetics, there has been the view that the Levinthal paradox&#x02014;of how a protein searches its conformational space so quickly&#x02014;might be explained by a folding mechanism, i.e., by some higher-level description (beyond the statement that it is stochastic) that clarifies how the protein decides which structures to form and avoids searching vast stretches of the conformational space in the first place.</p><p id="P32">Zipping and assembly (ZA) is a hypothesis for a general folding mechanism. On fast timescales, small fragments of the chain can search their conformations more completely than larger fragments can (<a href="#R53" rid="R53" class=" bibr popnode">53</a>, <a href="#R73" rid="R73" class=" bibr popnode">73</a>). There are certain problems of global optimization&#x02014;including the ZA mechanism of protein folding and the Cocke-Kasami-Younger method for parsing sentences (<a href="#R54" rid="R54" class=" bibr popnode">54</a>, <a href="#R102" rid="R102" class=" bibr popnode">102</a>)&#x02014;in which the globally optimal solution (native structure, in this case) can almost always be found (although not guaranteed) by a divide-and-conquer strategy, a fast process of cobbling together smaller locally optimal decisions. Accordingly, in the earliest time steps after folding is initiated (picoseconds to nanoseconds), each of the different peptide fragments of the chain searches for small local metastable structures, such as helical turns, &#x003b2;-turns, or small loops. Each peptide segment searches its own conformations, at the same time that other segments are searching. Not stable on their own, a few of those local structures are sufficiently metastable to survive to the next longer timescale, where they grow (or zip) into increasingly larger and more stable structures. On still longer timescales, pairs or groups of these substructures can assemble into structures that are still larger and more native-like, and metastability gives way to stability (<a href="#R97" rid="R97" class=" bibr popnode">97</a>, <a href="#R102" rid="R102" class=" bibr popnode">102</a>, <a href="#R103" rid="R103" class=" bibr popnode">103</a>, <a href="#R199" rid="R199" class=" bibr popnode">199</a>, <a href="#R215" rid="R215" class=" bibr popnode">215</a>, <a href="#R223" rid="R223" class=" bibr popnode">223</a>, <a href="#R228" rid="R228" class=" bibr popnode">228</a>&#x02013;<a href="#R230" rid="R230" class=" bibr popnode">230</a>, <a href="#R232" rid="R232" class=" bibr popnode">232</a>).</p><p id="P33" class="p p-last">The ZA mechanism shares much in common with other mechanisms, such as diffusion-collision, hierarchical, and foldon models. The last two mechanisms, however, are descriptors of experiments. They do not prescribe how to compute a protein&#x02019;s folding route from its amino acid sequence. In contrast, the ZA mechanism is such a microscopic recipe, starting from the amino acid sequence and specifying a time series of ensembles of conformations the chain searches at each stage of folding. ZA is a funnel process: There are many parallel microscopic routes at the beginning, and fewer and more sequential routes at the end. The ZA mechanism provides a plausible answer to Levinthal&#x02019;s paradox of what vast stretches of conformational space the protein never bothers to search. For any compact native polymer structure, there are always routes to the native state that take only small-conformational-entropy-loss steps. ZA otherwise explores very little of conformational space. These few routes constitute the dominant folding processes in the ZA mechanism. One test of this mechanism is the prediction of the change of folding routes (<a href="#R229" rid="R229" class=" bibr popnode">229</a>), measured by the change in &#x003c6;-value distributions (<a href="#R143" rid="R143" class=" bibr popnode">143</a>), upon circular permutation of the chain. Proteins can be circularly permuted if the chain termini are adjacent to each other in the wild-type native structure. In such cases, the ends are covalently linked and the chain is broken elsewhere. This alters the native topology (contact map) dramatically, and sometimes the folding routes, but does not appear to substantially change the native structure (<a href="#R20" rid="R20" class=" bibr popnode">20</a>, <a href="#R142" rid="R142" class=" bibr popnode">142</a>, <a href="#R143" rid="R143" class=" bibr popnode">143</a>, <a href="#R160" rid="R160" class=" bibr popnode">160</a>, <a href="#R221" rid="R221" class=" bibr popnode">221</a>).</p></div></div><div id="S19" class="tsec sec"><h2 class="head no_bottom_margin" id="S19title">PHYSICS-BASED MODELING OF FOLDING AND STRUCTURE PREDICTION</h2><p id="P34" class="p p-first">Computer simulations of purely physics-based models are becoming useful for structure prediction and for studying folding routes. Here the metric of success is not purely performance in native structure prediction; it is to gain a deeper understanding of the forces and dynamics that govern protein properties. When purely physical methods are successful, it will allow us to go beyond bioinformatics to (<em>a</em>) predict conformational changes, such as induced fit, important for computational drug discovery; (<em>b</em>) understand protein mechanisms of action, motions, folding processes, enzymatic catalysis, and other situations that require more than just the static native structure; (<em>c</em>) understand how proteins respond to solvents, pH, salts, denaturants, and other factors; and (<em>d</em>) design synthetic proteins having noncanonical amino acids or foldameric polymers with nonbiological backbones.</p><p id="P35">A key issue has been whether semiempirical atomic physical force fields are good enough to fold up a protein in a computer. Physics-based methods are currently limited by large computational requirements owing to the formidable conformational search problem and, to a lesser extent, by weaknesses in force fields. Nevertheless, there have been notable successes in the past decade enabled by the development of large supercomputer resources and distributed computing systems. The first milestone was a supercomputer simulation by Duan and Kollman in 1998 of the folding of the 36-residue villin headpiece in explicit solvent, for nearly a microsecond of computed time, reaching a collapsed state 4.5 &#x000c5; from the NMR structure (<a href="#R57" rid="R57" class=" bibr popnode">57</a>). Another milestone was the development by Pande and colleagues of Folding@home, a distributed grid computing system running on the screensavers of volunteer computers worldwide. Pande and colleagues (<a href="#R241" rid="R241" class=" bibr popnode">241</a>) have studied the folding kinetics of villin. High-resolution structures of villin have recently been reached by Pande and colleagues (<a href="#R110" rid="R110" class=" bibr popnode">110</a>) and Duan and colleagues (<a href="#R136" rid="R136" class=" bibr popnode">136</a>, <a href="#R137" rid="R137" class=" bibr popnode">137</a>). In addition, three groups have folded the 20-residue Trp-cage peptide to ~1 &#x000c5;: Simmerling et al. (<a href="#R200" rid="R200" class=" bibr popnode">200</a>), the IBM Blue Gene group of Pitera and Swope (<a href="#R183" rid="R183" class=" bibr popnode">183</a>), and Duan and colleagues (<a href="#R35" rid="R35" class=" bibr popnode">35</a>). Recently, Lei &#x00026; Duan (<a href="#R135" rid="R135" class=" bibr popnode">135</a>) folded the albumin-binding domain, a 47-residue, three-helix bundle, to 2.0 &#x000c5;. Physics-based approaches are also folding small helices and &#x003b2;-hairpin peptides of up to ~20 residues that have stable secondary structures (63, 81, 108, 240, 246; M.S. Shell, R. Ritterson &#x00026; K.A. Dill, unpublished data). Physical potential models have also been sampled using non-Boltzmann stochastic and deterministic optimization strategies (<a href="#R121" rid="R121" class=" bibr popnode">121</a>, <a href="#R174" rid="R174" class=" bibr popnode">174</a>, <a href="#R207" rid="R207" class=" bibr popnode">207</a>, <a href="#R220" rid="R220" class=" bibr popnode">220</a>).</p><p id="P36">Here are some of the key conclusions. First, a powerful way to sample conformations and obtain proper Boltzmann averages is replica exchange molecular dynamics (REMD) (<a href="#R210" rid="R210" class=" bibr popnode">210</a>). Second, although force fields are good, they need improvements in backbone torsional energies to address the balance between helical and extended conformations (<a href="#R81" rid="R81" class=" bibr popnode">81</a>, <a href="#R108" rid="R108" class=" bibr popnode">108</a>, <a href="#R240" rid="R240" class=" bibr popnode">240</a>), and in implicit solvation, which dramatically reduces the expense relative to explicit water simulations but which frequently overstabilizes ion-pairing interactions, in turn destabilizing native structures (<a href="#R63" rid="R63" class=" bibr popnode">63</a>, <a href="#R246" rid="R246" class=" bibr popnode">246</a>).</p><p id="P37">Can modern force fields with Boltzmann sampling predict larger native structures? Recent work indicates that, when combined with a conformational search technique based on the ZA folding mechanism, purely physics-based methods can arrive at structures close to the native state for chains up to ~100 monomers (177; M.S. Shell, S.B. Ozkan, V.A. Voelz, G.H.A. Wu &#x00026; K.A. Dill, unpublished data). The approach, called ZAM (zipping and assembly method), uses replica exchange and the AMBER96 force field and works by (<em>a</em>) breaking the full protein chain into small fragments (initially 8-mers), which are simulated separately using REMD; (<em>b</em>) then growing or zipping the fragments having metastable structures by adding a few new residues or assembling two such fragments together, with further REMD and iterations; and (<em>c</em>) locking in place any stable residue-residue contacts with a harmonic spring, enforcing emerging putative physical folding routes, without the need to sample huge numbers of degrees of freedom at a time.</p><p id="P38" class="p p-last">ZAM was tested through the folding of eight of nine small proteins from the PDB to within 2.5 &#x000c5;, using a 70-processor cluster over 6 months (<a href="#R177" rid="R177" class=" bibr popnode">177</a>), giving good agreement with the &#x003c6;-values known for four of them. <a href="/pmc/articles/PMC2443096/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9</span></a> shows the ZAM folding process for one of these proteins, and <a href="/pmc/articles/PMC2443096/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 10</span></a> shows the predicted versus experimental structures for all nine. In a more stringent test, ZAM was applied in CASP7 to the folding of six small proteins from 76 to 112 residues (M.S. Shell, S.B. Ozkan, V.A. Voelz, G.H.A. Wu &#x00026; K.A. Dill, unpublished data). Of the four proteins attempted in CASP7 that were not domain-swapped, ZAM predicted roughly correct tertiary structures, segments of more than 40 residues with an average RMSD of 5.9 angstroms, and secondary structures with 73% accuracy. From these studies it has been concluded that ZA routes can identify limited-sampling routes to the native state from unfolded states, directed by all-atom force fields, and that the AMBER96 plus a generalized Born implicit solvent model is a reasonable scoring function. Fragments that adopt incorrect secondary structures early in the simulations are frequently corrected in later-stage folding because the emerging tertiary structure of the protein often will not tolerate them.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F9" co-legend-rid="lgnd_F9"><a href="/pmc/articles/PMC2443096/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787312897616"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f9.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f9.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f9.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787312897616"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Figure 9</a></div><!--caption a7--><div class="caption"><p>The folding routes found in the ZAM conformational search process for protein G, from the work described in Reference <a href="#R177" rid="R177" class=" bibr popnode">177</a>. The chain is first parsed into many short, overlapping fragments. After sampling by replica exchange molecular dynamics, stable hydrophobic contacts are identified and restrained. Fragments are then either (<em>a</em>) grown or zipped though iterations of adding new residues, sampling, and contact detection, or (<em>b</em>) assembled together pairwise using rigid body alignment followed by further sampling until a completed structure is reached.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F10" co-legend-rid="lgnd_F10"><a href="/pmc/articles/PMC2443096/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139787297175120"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f10.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms51627f10.jpg" src="/pmc/articles/PMC2443096/bin/nihms51627f10.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm139787297175120"><a target="object" rel="noopener" href="/pmc/articles/PMC2443096/figure/F10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="/pmc/articles/PMC2443096/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Figure 10</a></div><!--caption a7--><div class="caption"><p>Ribbon diagrams of the predicted protein structures using the ZAM search algorithm (<em>purple</em>) versus experimental PDB structures (<em>orange</em>). The backbone C-RMSDs with respect to PDB structures are protein A (1.9 &#x000c5;), albumin domain binding protein (2.4 &#x000c5;), alpha-3D [2.85 &#x000c5; (excluding loop residues) or 4.6 &#x000c5;], FBP26 WW domain (2.2 &#x000c5;), YJQ8 WW domain (2.0 &#x000c5;), 1&#x02013;35 residue fragment of Ubiquitin (2.0 &#x000c5;), protein G (1.6 &#x000c5;), and -spectrin SH3 (2.2 &#x000c5;). ZAM fails to find the src-SH3 structure: Shown is a conformation that is 6 &#x000c5; from the experimental structure. The problem in this case appears to be overstabilization of nonnative ion pairs in the GB/SA implicit solvation model.</p></div></div></div></div><div id="S20" class="tsec sec"><h2 class="head no_bottom_margin" id="S20title">SUMMARY</h2><p id="P39" class="p p-first">The protein folding problem has seen enormous advances over the last fifty years. New experimental techniques have arisen, including hydrogen exchange, &#x003c6;-value methods that probe mutational effects on folding rates, single-molecule methods that can explore heterogeneity of folding and energy landscapes, and fast temperature-jump methods. New theoretical and computational approaches have emerged, including methods of bioinformatics, multiple-sequence alignments, structure-prediction Web servers, physics-based force fields of good accuracy, physical models of energy landscapes, fast methods of conformational sampling and searching, master-equation methods to explore the physical mechanisms of folding, parallel and distributed grid-based computing, and the CASP community-wide event for protein structure prediction.</p><p id="P40" class="p">Protein folding no longer appears to be the insurmountable grand challenge that it once appeared to be. Current knowledge of folding codes is sufficient to guide the successful designs of new proteins and foldameric materials. For the once seemingly intractable Levinthal puzzle, there is now a viable hypothesis: A protein can fold quickly and solve its big global optimization puzzle by piecewise solutions of smaller component puzzles. Other matters of principle are now yielding to theory and physics-based modeling. And current computer algorithms are now predicting native structures of small proteins remarkably accurately, promising growing value in drug discovery and proteomics.</p><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="sec" id="S21"><h3 id="S21title">SUMMARY POINTS</h3><ol class="enumerated" style="list-style-type:decimal" id="L1"><li><div id="P41">The protein folding code is mainly embodied in side chain solvation interactions. Novel protein folds and nonbiological foldamers are now being successfully designed and are moving toward practical applications.</div></li><li><div id="P42">Thanks to CASP, the growing PDB, and fast-homology and sequence alignment methods, computer methods now can often predict correct native structures of small proteins.</div></li><li><div id="P43">The protein folding problem has both driven&#x02014;and benefited from&#x02014;big advances in experimental and theoretical/computational methods.</div></li><li><div id="P44">Proteins fold on funnel-shaped energy landscapes, which describe the conformational heterogeneity among the nonnative states. This heterogeneity is key to the entropy that opposes folding and thus to folding equilibria. This heterogeneity is also important for understanding folding kinetics at the level of the individual chain processes.</div></li><li><div id="P45">A protein can fold quickly to its native structure by ZA, making independent local decisions first and then combining those substructures. In this way, a protein can avoid searching most of its conformational space. ZA appears to be a useful search method for computational modeling.</div></li></ol></div></div></div><div id="S23" class="tsec sec"><h2 class="head no_bottom_margin" id="S23title">Acknowledgments</h2><div class="sec"><p>For very helpful comments and insights, both on this review and through ongoing discussions over the years, we are deeply grateful to D. Wayne Bolen, Hue Sun Chan, John Chodera, Yong Duan, Walter Englander, Frank Noe, Jos&#x000e9; Onuchic, Vijay Pande, Jed Pitera, Kevin Plaxco, Adrian Roitberg, George Rose, Tobin Sosnick, Bill Swope, Dave Thirumalai, Vince Voelz, Peter G Wolynes, and Huan-Xiang Zhou. We owe particular thanks and appreciation to Buzz Baldwin, to whom this volume of the <em>Annual Review of Biophysics</em> is dedicated, not only for his interest and engagement with us on matters of protein folding over the many years, but also for his pioneering and founding leadership of the whole field. We appreciate the support from NIH grant GM 34993, the Air Force, and the Sandler Foundation.</p></div></div><div id="fn-group-a.n.c" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.n.ctitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><p class="p p-first">DISCLOSURE STATEMENT</p><p class="p p-last">The authors are not aware of any biases that might be perceived as affecting the objectivity of this review.</p></p></div></div><div id="ref-list-a.n.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.n.dtitle">LITERATURE CITED</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">So much more to know. . . <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">309</span>:78–102.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15994524" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=So+much+more+to+know.+.+.&amp;volume=309&amp;publication_year=2005&amp;pages=78-102&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Allen F, Coteus P, Crumley P, Curioni A, Denneau M, et al. Blue gene: a vision for protein science using a petaflop supercomputer. <span><span class="ref-journal">IBM Syst J. </span>2001;<span class="ref-vol">40</span>:310–27.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IBM+Syst+J&amp;title=Blue+gene:+a+vision+for+protein+science+using+a+petaflop+supercomputer&amp;author=F+Allen&amp;author=P+Coteus&amp;author=P+Crumley&amp;author=A+Curioni&amp;author=M+Denneau&amp;volume=40&amp;publication_year=2001&amp;pages=310-27&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Anfinsen CB. Principles that govern the folding of protein chains. <span><span class="ref-journal">Science. </span>1973;<span class="ref-vol">181</span>:223–30.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4124164" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Principles+that+govern+the+folding+of+protein+chains&amp;author=CB+Anfinsen&amp;volume=181&amp;publication_year=1973&amp;pages=223-30&amp;pmid=4124164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Anfinsen CB, Scheraga HA. Experimental and theoretical aspects of protein folding. <span><span class="ref-journal">Adv Protein Chem. </span>1975;<span class="ref-vol">29</span>:205–300.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/237413" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Protein+Chem&amp;title=Experimental+and+theoretical+aspects+of+protein+folding&amp;author=CB+Anfinsen&amp;author=HA+Scheraga&amp;volume=29&amp;publication_year=1975&amp;pages=205-300&amp;pmid=237413&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Auton M, Holthauzen LM, Bolen DW. Anatomy of energetic changes accompanying urea-induced protein denaturation. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:15317–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2000523/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17878304" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Anatomy+of+energetic+changes+accompanying+urea-induced+protein+denaturation&amp;author=M+Auton&amp;author=LM+Holthauzen&amp;author=DW+Bolen&amp;volume=104&amp;publication_year=2007&amp;pages=15317-22&amp;pmid=17878304&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Avbelj F, Baldwin RL. Role of backbone solvation in determining thermodynamic &#x003b2; propensities of the amino acids. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:1309–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122186/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11805303" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Role+of+backbone+solvation+in+determining+thermodynamic+&#x003b2;+propensities+of+the+amino+acids&amp;author=F+Avbelj&amp;author=RL+Baldwin&amp;volume=99&amp;publication_year=2002&amp;pages=1309-13&amp;pmid=11805303&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Bachmann A, Kiefhaber T. Apparent two-state tendamistat folding is a sequential process along a defined route. <span><span class="ref-journal">J Mol Biol. </span>2001;<span class="ref-vol">306</span>:375–86.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11237606" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Apparent+two-state+tendamistat+folding+is+a+sequential+process+along+a+defined+route&amp;author=A+Bachmann&amp;author=T+Kiefhaber&amp;volume=306&amp;publication_year=2001&amp;pages=375-86&amp;pmid=11237606&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Baker D. Prediction and design of macromolecular structures and interactions. <span><span class="ref-journal">Philos Trans R Soc B Biol Sci. </span>2006;<span class="ref-vol">361</span>:459–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1609347/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16524834" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos+Trans+R+Soc+B+Biol+Sci&amp;title=Prediction+and+design+of+macromolecular+structures+and+interactions&amp;author=D+Baker&amp;volume=361&amp;publication_year=2006&amp;pages=459-63&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Baker D, Sali A. Protein structure prediction and structural genomics. <span><span class="ref-journal">Science. </span>2001;<span class="ref-vol">294</span>:93–96.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11588250" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Protein+structure+prediction+and+structural+genomics&amp;author=D+Baker&amp;author=A+Sali&amp;volume=294&amp;publication_year=2001&amp;pages=93-96&amp;pmid=11588250&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Baldwin RL, Rose GD. Is protein folding hierarchic? I. Local structure and peptide folding. <span><span class="ref-journal">Trends Biochem Sci. </span>1999;<span class="ref-vol">24</span>:26–33.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10087919" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem+Sci&amp;title=Is+protein+folding+hierarchic?+I.+Local+structure+and+peptide+folding&amp;author=RL+Baldwin&amp;author=GD+Rose&amp;volume=24&amp;publication_year=1999&amp;pages=26-33&amp;pmid=10087919&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Banavar JR, Maritan A. Physics of proteins. <span><span class="ref-journal">Annu Rev Biophys Biomol Struct. </span>2007;<span class="ref-vol">36</span>:261–80.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17477839" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biophys+Biomol+Struct&amp;title=Physics+of+proteins&amp;author=JR+Banavar&amp;author=A+Maritan&amp;volume=36&amp;publication_year=2007&amp;pages=261-80&amp;pmid=17477839&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Banavar JR, Maritan A, Micheletti C, Trovato A. Geometry and physics of proteins. <span><span class="ref-journal">Proteins. </span>2002;<span class="ref-vol">47</span>:315–22.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11948785" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Geometry+and+physics+of+proteins&amp;author=JR+Banavar&amp;author=A+Maritan&amp;author=C+Micheletti&amp;author=A+Trovato&amp;volume=47&amp;publication_year=2002&amp;pages=315-22&amp;pmid=11948785&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Best RB, Hummer G. Reaction coordinates and rates from transition paths. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:6732–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1100744/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15814618" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Reaction+coordinates+and+rates+from+transition+paths&amp;author=RB+Best&amp;author=G+Hummer&amp;volume=102&amp;publication_year=2005&amp;pages=6732-37&amp;pmid=15814618&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Bieri O, Wirz J, Hellrung B, Schutkowski M, Drewello M, Kiefhaber T. The speed limit for protein folding measured by triplet-triplet energy transfer. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:9597–601.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC22254/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10449738" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+speed+limit+for+protein+folding+measured+by+triplet-triplet+energy+transfer&amp;author=O+Bieri&amp;author=J+Wirz&amp;author=B+Hellrung&amp;author=M+Schutkowski&amp;author=M+Drewello&amp;volume=96&amp;publication_year=1999&amp;pages=9597-601&amp;pmid=10449738&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Bolhuis PG. Kinetic pathways of &#x003b2;-hairpin (un)folding in explicit solvent. <span><span class="ref-journal">Biophys J. </span>2005;<span class="ref-vol">88</span>:50–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1305027/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15516524" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=Kinetic+pathways+of+&#x003b2;-hairpin+(un)folding+in+explicit+solvent&amp;author=PG+Bolhuis&amp;volume=88&amp;publication_year=2005&amp;pages=50-61&amp;pmid=15516524&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Bradley CM, Barrick D. The Notch ankyrin domain folds via a discrete, centralized pathway. <span><span class="ref-journal">Structure. </span>2006;<span class="ref-vol">14</span>:1303–12.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16905104" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=The+Notch+ankyrin+domain+folds+via+a+discrete,+centralized+pathway&amp;author=CM+Bradley&amp;author=D+Barrick&amp;volume=14&amp;publication_year=2006&amp;pages=1303-12&amp;pmid=16905104&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Bradley P, Misura KMS, Baker D. Toward high-resolution de novo structure prediction for small proteins. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">309</span>:1868–71.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16166519" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Toward+high-resolution+de+novo+structure+prediction+for+small+proteins&amp;author=P+Bradley&amp;author=KMS+Misura&amp;author=D+Baker&amp;volume=309&amp;publication_year=2005&amp;pages=1868-71&amp;pmid=16166519&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Brant DA, Flory PJ. The role of dipole interactions in determining polypeptide configurations. <span><span class="ref-journal">J Am Chem Soc. </span>1965;<span class="ref-vol">87</span>:663–64.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=The+role+of+dipole+interactions+in+determining+polypeptide+configurations&amp;author=DA+Brant&amp;author=PJ+Flory&amp;volume=87&amp;publication_year=1965&amp;pages=663-64&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Bryngelson JD, Wolynes PG. Spin glasses and the statistical mechanics of protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1987;<span class="ref-vol">84</span>:7524–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC299331/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3478708" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Spin+glasses+and+the+statistical+mechanics+of+protein+folding&amp;author=JD+Bryngelson&amp;author=PG+Wolynes&amp;volume=84&amp;publication_year=1987&amp;pages=7524-28&amp;pmid=3478708&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Bulaj G, Koehn RE, Goldenberg DP. Alteration of the disulfide-coupled folding pathway of BPTI by circular permutation. <span><span class="ref-journal">Protein Sci. </span>2004;<span class="ref-vol">13</span>:1182–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2286756/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15096625" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Alteration+of+the+disulfide-coupled+folding+pathway+of+BPTI+by+circular+permutation&amp;author=G+Bulaj&amp;author=RE+Koehn&amp;author=DP+Goldenberg&amp;volume=13&amp;publication_year=2004&amp;pages=1182-96&amp;pmid=15096625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Byrne MP, Manuel RL, Lowe LG, Stites WE. Energetic contribution of side chain hydrogen bonding to the stability of staphylococcal nuclease. <span><span class="ref-journal">Biochemistry. </span>1995;<span class="ref-vol">34</span>:13949–60.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7577991" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Energetic+contribution+of+side+chain+hydrogen+bonding+to+the+stability+of+staphylococcal+nuclease&amp;author=MP+Byrne&amp;author=RL+Manuel&amp;author=LG+Lowe&amp;author=WE+Stites&amp;volume=34&amp;publication_year=1995&amp;pages=13949-60&amp;pmid=7577991&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Callender RH, Dyer RB, Gilmanshin R, Woodruff WH. Fast events in protein folding: the time evolution of primary processes. <span><span class="ref-journal">Annu Rev Phys Chem. </span>1998;<span class="ref-vol">49</span>:173–202.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9933907" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Phys+Chem&amp;title=Fast+events+in+protein+folding:+the+time+evolution+of+primary+processes&amp;author=RH+Callender&amp;author=RB+Dyer&amp;author=R+Gilmanshin&amp;author=WH+Woodruff&amp;volume=49&amp;publication_year=1998&amp;pages=173-202&amp;pmid=9933907&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Cecconi C, Shank EA, Bustamante C, Marqusee S. Direct observation of the three-state folding of a single protein molecule. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">309</span>:2057–60.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16179479" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Direct+observation+of+the+three-state+folding+of+a+single+protein+molecule&amp;author=C+Cecconi&amp;author=EA+Shank&amp;author=C+Bustamante&amp;author=S+Marqusee&amp;volume=309&amp;publication_year=2005&amp;pages=2057-60&amp;pmid=16179479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Cellitti J, Bernstein R, Marqusee S. Exploring subdomain cooperativity in T4 lysozyme. II. Uncovering the C-terminal subdomain as a hidden intermediate in the kinetic folding pathway. <span><span class="ref-journal">Protein Sci. </span>2007;<span class="ref-vol">16</span>:852–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2206637/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17400925" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Exploring+subdomain+cooperativity+in+T4+lysozyme.+II.+Uncovering+the+C-terminal+subdomain+as+a+hidden+intermediate+in+the+kinetic+folding+pathway&amp;author=J+Cellitti&amp;author=R+Bernstein&amp;author=S+Marqusee&amp;volume=16&amp;publication_year=2007&amp;pages=852-62&amp;pmid=17400925&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Chan HS, Dill KA. Origins of structure in globular proteins. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1990;<span class="ref-vol">87</span>:6388–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC54539/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2385597" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Origins+of+structure+in+globular+proteins&amp;author=HS+Chan&amp;author=KA+Dill&amp;volume=87&amp;publication_year=1990&amp;pages=6388-92&amp;pmid=2385597&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Chan HS, Dill KA. Transition states and folding dynamics of proteins and heteropolymers. <span><span class="ref-journal">J Chem Phys. </span>1994;<span class="ref-vol">100</span>:9238–57.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Transition+states+and+folding+dynamics+of+proteins+and+heteropolymers&amp;author=HS+Chan&amp;author=KA+Dill&amp;volume=100&amp;publication_year=1994&amp;pages=9238-57&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Chekmarev SF, Krivov SV, Karplus M. Folding time distributions as an approach to protein folding kinetics. <span><span class="ref-journal">J Phys Chem B. </span>2005;<span class="ref-vol">109</span>:5312–30.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16863198" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Folding+time+distributions+as+an+approach+to+protein+folding+kinetics&amp;author=SF+Chekmarev&amp;author=SV+Krivov&amp;author=M+Karplus&amp;volume=109&amp;publication_year=2005&amp;pages=5312-30&amp;pmid=16863198&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Chen J, Stites WE. Packing is a key selection factor in the evolution of protein hydrophobic cores. <span><span class="ref-journal">Biochemistry. </span>2001;<span class="ref-vol">40</span>:15280–89.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11735410" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Packing+is+a+key+selection+factor+in+the+evolution+of+protein+hydrophobic+cores&amp;author=J+Chen&amp;author=WE+Stites&amp;volume=40&amp;publication_year=2001&amp;pages=15280-89&amp;pmid=11735410&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Chikenji G, Fujitsuka Y, Takada S. Shaping up the protein folding funnel by local interaction: lesson from a structure prediction study. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:3141–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1413881/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16488978" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Shaping+up+the+protein+folding+funnel+by+local+interaction:+lesson+from+a+structure+prediction+study&amp;author=G+Chikenji&amp;author=Y+Fujitsuka&amp;author=S+Takada&amp;volume=103&amp;publication_year=2006&amp;pages=3141-46&amp;pmid=16488978&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Cho SS, Levy Y, Wolynes PG. P versus Q: structural reaction coordinates capture protein folding on smooth landscapes. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:586–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1334664/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16407126" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=P+versus+Q:+structural+reaction+coordinates+capture+protein+folding+on+smooth+landscapes&amp;author=SS+Cho&amp;author=Y+Levy&amp;author=PG+Wolynes&amp;volume=103&amp;publication_year=2006&amp;pages=586-91&amp;pmid=16407126&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Chodera JD, Singhal N, Pande VS, Dill KA, Swope WC. Automatic discovery of metastable states for the construction of Markov models of macromolecular conformational dynamics. <span><span class="ref-journal">J Chem Phys. </span>2007;<span class="ref-vol">126</span>:155101.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17461665" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Automatic+discovery+of+metastable+states+for+the+construction+of+Markov+models+of+macromolecular+conformational+dynamics&amp;author=JD+Chodera&amp;author=N+Singhal&amp;author=VS+Pande&amp;author=KA+Dill&amp;author=WC+Swope&amp;volume=126&amp;publication_year=2007&amp;pages=155101&amp;pmid=17461665&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Chothia C, Lesk AM. The relation between the divergence of sequence and structure in proteins. <span><span class="ref-journal">EMBO J. </span>1986;<span class="ref-vol">5</span>:823–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1166865/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3709526" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=The+relation+between+the+divergence+of+sequence+and+structure+in+proteins&amp;author=C+Chothia&amp;author=AM+Lesk&amp;volume=5&amp;publication_year=1986&amp;pages=823-26&amp;pmid=3709526&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Chou PY, Fasman GD. Prediction of protein conformation. <span><span class="ref-journal">Biochemistry. </span>1974;<span class="ref-vol">13</span>:222–45.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/4358940" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Prediction+of+protein+conformation&amp;author=PY+Chou&amp;author=GD+Fasman&amp;volume=13&amp;publication_year=1974&amp;pages=222-45&amp;pmid=4358940&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Chou PY, Fasman GD. Empirical predictions of protein conformation. <span><span class="ref-journal">Annu Rev Biochem. </span>1978;<span class="ref-vol">47</span>:251–76.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/354496" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biochem&amp;title=Empirical+predictions+of+protein+conformation&amp;author=PY+Chou&amp;author=GD+Fasman&amp;volume=47&amp;publication_year=1978&amp;pages=251-76&amp;pmid=354496&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Chowdhury S, Lee MC, Xiong G, Duan Y. Ab initio folding simulation of the Trp-cage mini-protein approaches NMR resolution. <span><span class="ref-journal">J Mol Biol. </span>2003;<span class="ref-vol">327</span>:711–17.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12634063" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Ab+initio+folding+simulation+of+the+Trp-cage+mini-protein+approaches+NMR+resolution&amp;author=S+Chowdhury&amp;author=MC+Lee&amp;author=G+Xiong&amp;author=Y+Duan&amp;volume=327&amp;publication_year=2003&amp;pages=711-17&amp;pmid=12634063&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Cieplak M, Henkel M, Karbowski J, Banavar JR. Master equation approach to protein folding and kinetic traps. <span><span class="ref-journal">Phys Rev Lett. </span>1998;<span class="ref-vol">80</span>:3654–57.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys+Rev+Lett&amp;title=Master+equation+approach+to+protein+folding+and+kinetic+traps&amp;author=M+Cieplak&amp;author=M+Henkel&amp;author=J+Karbowski&amp;author=JR+Banavar&amp;volume=80&amp;publication_year=1998&amp;pages=3654-57&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Cieplak M, Xuan Hoang T. Scaling of folding properties in Go models of proteins. <span><span class="ref-journal">J Biol Phys. </span>2000;<span class="ref-vol">26</span>:273–94.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3456313/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23345727" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Phys&amp;title=Scaling+of+folding+properties+in+Go+models+of+proteins&amp;author=M+Cieplak&amp;author=T+Xuan+Hoang&amp;volume=26&amp;publication_year=2000&amp;pages=273-94&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Clementi C, Nymeyer H, Onuchic JN. Topological and energetic factors: What determines the structural details of the transition state ensemble and &#x0201c;en-route&#x0201d; intermediates for protein folding? An investigation for small globular proteins. <span><span class="ref-journal">J Mol Biol. </span>2000;<span class="ref-vol">298</span>:937–53.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10801360" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Topological+and+energetic+factors:+What+determines+the+structural+details+of+the+transition+state+ensemble+and+&#x0201c;en-route&#x0201d;+intermediates+for+protein+folding?+An+investigation+for+small+globular+proteins&amp;author=C+Clementi&amp;author=H+Nymeyer&amp;author=JN+Onuchic&amp;volume=298&amp;publication_year=2000&amp;pages=937-53&amp;pmid=10801360&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Cordes MHJ, Davidsont AR, Sauer RT. Sequence space, folding and protein design. <span><span class="ref-journal">Curr Opin Struct Biol. </span>1996;<span class="ref-vol">6</span>:3–10.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8696970" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Sequence+space,+folding+and+protein+design&amp;author=MHJ+Cordes&amp;author=AR+Davidsont&amp;author=RT+Sauer&amp;volume=6&amp;publication_year=1996&amp;pages=3-10&amp;pmid=8696970&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Cornell WD, Cieplak P, Bayly CI, Gould IR, Merz KM, et al. A second generation force field for the simulation of proteins, nucleic acids, and organic molecules. <span><span class="ref-journal">J Am Chem Soc. </span>1995;<span class="ref-vol">117</span>:5179–97.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=A+second+generation+force+field+for+the+simulation+of+proteins,+nucleic+acids,+and+organic+molecules&amp;author=WD+Cornell&amp;author=P+Cieplak&amp;author=CI+Bayly&amp;author=IR+Gould&amp;author=KM+Merz&amp;volume=117&amp;publication_year=1995&amp;pages=5179-97&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Creamer TP, Rose GD. A-helix-forming propensities in peptides and proteins. <span><span class="ref-journal">Proteins. </span>1994;<span class="ref-vol">19</span>:85–97.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8090712" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A-helix-forming+propensities+in+peptides+and+proteins&amp;author=TP+Creamer&amp;author=GD+Rose&amp;volume=19&amp;publication_year=1994&amp;pages=85-97&amp;pmid=8090712&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Cymes GD, Grosman C, Auerbach A. Structure of the transition state of gating in the acetylcholine receptor channel pore: a &#x003c6;-value analysis. <span><span class="ref-journal">Biochemistry. </span>2002;<span class="ref-vol">41</span>:5548–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6442467/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11969415" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structure+of+the+transition+state+of+gating+in+the+acetylcholine+receptor+channel+pore:+a+&#x003c6;-value+analysis&amp;author=GD+Cymes&amp;author=C+Grosman&amp;author=A+Auerbach&amp;volume=41&amp;publication_year=2002&amp;pages=5548-55&amp;pmid=11969415&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Dahiyat BI, Mayo SL. De novo protein design: fully automated sequence selection. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">278</span>:82–87.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9311930" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=De+novo+protein+design:+fully+automated+sequence+selection&amp;author=BI+Dahiyat&amp;author=SL+Mayo&amp;volume=278&amp;publication_year=1997&amp;pages=82-87&amp;pmid=9311930&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="citation">de Los Rios MA, Daneshi M, Plaxco KW. Experimental investigation of the frequency and substitution dependence of negative &#x003c6;-values in two-state proteins. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:12160–67.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16142914" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Experimental+investigation+of+the+frequency+and+substitution+dependence+of+negative+&#x003c6;-values+in+two-state+proteins&amp;author=MA+de+Los+Rios&amp;author=M+Daneshi&amp;author=KW+Plaxco&amp;volume=44&amp;publication_year=2005&amp;pages=12160-67&amp;pmid=16142914&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="citation">Debe DA, Carlson MJ, Goddard WA. The topomer-sampling model of protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:2596–601.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC15813/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10077555" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+topomer-sampling+model+of+protein+folding&amp;author=DA+Debe&amp;author=MJ+Carlson&amp;author=WA+Goddard&amp;volume=96&amp;publication_year=1999&amp;pages=2596-601&amp;pmid=10077555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="citation">Deechongkit S, Dawson PE, Kelly JW. Toward assessing the position-dependent contributions of backbone hydrogen bonding to &#x003b2;-sheet folding thermodynamics employing amide-to-ester perturbations. <span><span class="ref-journal">J Am Chem Soc. </span>2004;<span class="ref-vol">126</span>:16762–71.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15612714" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Toward+assessing+the+position-dependent+contributions+of+backbone+hydrogen+bonding+to+&#x003b2;-sheet+folding+thermodynamics+employing+amide-to-ester+perturbations&amp;author=S+Deechongkit&amp;author=PE+Dawson&amp;author=JW+Kelly&amp;volume=126&amp;publication_year=2004&amp;pages=16762-71&amp;pmid=15612714&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="citation">Dill KA. Theory for the folding and stability of globular proteins. <span><span class="ref-journal">Biochemistry. </span>1985;<span class="ref-vol">24</span>:1501–9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3986190" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Theory+for+the+folding+and+stability+of+globular+proteins&amp;author=KA+Dill&amp;volume=24&amp;publication_year=1985&amp;pages=1501-9&amp;pmid=3986190&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="citation">Dill KA. Dominant forces in protein folding. <span><span class="ref-journal">Biochemistry. </span>1990;<span class="ref-vol">29</span>:7133–55.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2207096" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Dominant+forces+in+protein+folding&amp;author=KA+Dill&amp;volume=29&amp;publication_year=1990&amp;pages=7133-55&amp;pmid=2207096&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="citation">Dill KA. Polymer principles and protein folding. <span><span class="ref-journal">Protein Sci. </span>1999;<span class="ref-vol">8</span>:1166–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2144345/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10386867" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Polymer+principles+and+protein+folding&amp;author=KA+Dill&amp;volume=8&amp;publication_year=1999&amp;pages=1166-80&amp;pmid=10386867&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="citation">Dill KA, Alonso DOV, Hutchinson K. Thermal stabilities of globular proteins. <span><span class="ref-journal">Biochemistry. </span>1989;<span class="ref-vol">28</span>:5439–49.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2775715" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Thermal+stabilities+of+globular+proteins&amp;author=KA+Dill&amp;author=DOV+Alonso&amp;author=K+Hutchinson&amp;volume=28&amp;publication_year=1989&amp;pages=5439-49&amp;pmid=2775715&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="citation">Dill KA, Bromberg S, Yue KZ, Fiebig KM, Yee DP, et al. Principles of protein folding: a perspective from simple exact models. <span><span class="ref-journal">Protein Sci. </span>1995;<span class="ref-vol">4</span>:561–602.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2143098/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7613459" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Principles+of+protein+folding:+a+perspective+from+simple+exact+models&amp;author=KA+Dill&amp;author=S+Bromberg&amp;author=KZ+Yue&amp;author=KM+Fiebig&amp;author=DP+Yee&amp;volume=4&amp;publication_year=1995&amp;pages=561-602&amp;pmid=7613459&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="citation">Dill KA, Chan HS. From Levinthal to pathways to funnels. <span><span class="ref-journal">Nat Struct Biol. </span>1997;<span class="ref-vol">4</span>:10–19.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8989315" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=From+Levinthal+to+pathways+to+funnels&amp;author=KA+Dill&amp;author=HS+Chan&amp;volume=4&amp;publication_year=1997&amp;pages=10-19&amp;pmid=8989315&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="citation">Dill KA, Fiebig KM, Chan HS. Cooperativity in protein-folding kinetics. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1993;<span class="ref-vol">90</span>:1942–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC45996/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7680482" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Cooperativity+in+protein-folding+kinetics&amp;author=KA+Dill&amp;author=KM+Fiebig&amp;author=HS+Chan&amp;volume=90&amp;publication_year=1993&amp;pages=1942-46&amp;pmid=7680482&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="citation">Dill KA, Lucas A, Hockenmaier J, Huang L, Chiang D, Joshi AK. Computational linguistics: a new tool for exploring biopolymer structures and statistical mechanics. <span><span class="ref-journal">Polymer. </span>2007;<span class="ref-vol">48</span>:4289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Polymer&amp;title=Computational+linguistics:+a+new+tool+for+exploring+biopolymer+structures+and+statistical+mechanics&amp;author=KA+Dill&amp;author=A+Lucas&amp;author=J+Hockenmaier&amp;author=L+Huang&amp;author=D+Chiang&amp;volume=48&amp;publication_year=2007&amp;pages=4289-300&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="citation">Drozdov AN, Grossfield A, Pappu RV. Role of solvent in determining conformational preferences of alanine dipeptide in water. <span><span class="ref-journal">J Am Chem Soc. </span>2004;<span class="ref-vol">126</span>:2574–81.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14982467" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Role+of+solvent+in+determining+conformational+preferences+of+alanine+dipeptide+in+water&amp;author=AN+Drozdov&amp;author=A+Grossfield&amp;author=RV+Pappu&amp;volume=126&amp;publication_year=2004&amp;pages=2574-81&amp;pmid=14982467&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="citation">Du R, Pande VS, Grosberg AY, Tanaka T, Shakhnovich ES. On the transition coordinate for protein folding. <span><span class="ref-journal">J Chem Phys. </span>1998;<span class="ref-vol">108</span>:334–50.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=On+the+transition+coordinate+for+protein+folding&amp;author=R+Du&amp;author=VS+Pande&amp;author=AY+Grosberg&amp;author=T+Tanaka&amp;author=ES+Shakhnovich&amp;volume=108&amp;publication_year=1998&amp;pages=334-50&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="citation">Duan Y, Kollman PA. Pathways to a protein folding intermediate observed in a 1-microsecond simulation in aqueous solution. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">282</span>:740–44.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9784131" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Pathways+to+a+protein+folding+intermediate+observed+in+a+1-microsecond+simulation+in+aqueous+solution&amp;author=Y+Duan&amp;author=PA+Kollman&amp;volume=282&amp;publication_year=1998&amp;pages=740-44&amp;pmid=9784131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="citation">Dyson HJ, Wright PE, Scheraga HA. The role of hydrophobic interactions in initiation and propagation of protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:13057–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1559752/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16916929" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+role+of+hydrophobic+interactions+in+initiation+and+propagation+of+protein+folding&amp;author=HJ+Dyson&amp;author=PE+Wright&amp;author=HA+Scheraga&amp;volume=103&amp;publication_year=2006&amp;pages=13057-61&amp;pmid=16916929&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="citation">Ellison PA, Cavagnero S. Role of unfolded state heterogeneity and en-route ruggedness in protein folding kinetics. <span><span class="ref-journal">Protein Sci. </span>2006;<span class="ref-vol">15</span>:564–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2249777/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16501227" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Role+of+unfolded+state+heterogeneity+and+en-route+ruggedness+in+protein+folding+kinetics&amp;author=PA+Ellison&amp;author=S+Cavagnero&amp;volume=15&amp;publication_year=2006&amp;pages=564-82&amp;pmid=16501227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="citation">Elmer SP, Park S, Pande VS. Foldamer dynamics expressed via Markov state models. I. Explicit solvent molecular-dynamics simulations in acetonitrile, chloroform, methanol, and water. <span><span class="ref-journal">J Chem Phys. </span>2005;<span class="ref-vol">123</span>:114902.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16392592" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Foldamer+dynamics+expressed+via+Markov+state+models.+I.+Explicit+solvent+molecular-dynamics+simulations+in+acetonitrile,+chloroform,+methanol,+and+water&amp;author=SP+Elmer&amp;author=S+Park&amp;author=VS+Pande&amp;volume=123&amp;publication_year=2005&amp;pages=114902&amp;pmid=16392592&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="citation">Elmer SP, Park S, Pande VS. Foldamer dynamics expressed via Markov state models. II. State space decomposition. <span><span class="ref-journal">J Chem Phys. </span>2005;<span class="ref-vol">123</span>:114903.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16392593" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Foldamer+dynamics+expressed+via+Markov+state+models.+II.+State+space+decomposition&amp;author=SP+Elmer&amp;author=S+Park&amp;author=VS+Pande&amp;volume=123&amp;publication_year=2005&amp;pages=114903&amp;pmid=16392593&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="citation">English EP, Chumanov RS, Gellman SH, Compton T. Rational development of &#x003b2;-peptide inhibitors of human cytomegalovirus entry. <span><span class="ref-journal">J Biol Chem. </span>2006;<span class="ref-vol">281</span>:2661–67.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16275647" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Rational+development+of+&#x003b2;-peptide+inhibitors+of+human+cytomegalovirus+entry&amp;author=EP+English&amp;author=RS+Chumanov&amp;author=SH+Gellman&amp;author=T+Compton&amp;volume=281&amp;publication_year=2006&amp;pages=2661-67&amp;pmid=16275647&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="citation">Felts AK, Harano Y, Gallicchio E, Levy RM. Free energy surfaces of &#x003b2;-hairpin and &#x003b1;-helical peptides generated by replica exchange molecular dynamics with the AGBNP implicit solvent model. <span><span class="ref-journal">Proteins. </span>2004;<span class="ref-vol">56</span>:310–21.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15211514" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Free+energy+surfaces+of+&#x003b2;-hairpin+and+&#x003b1;-helical+peptides+generated+by+replica+exchange+molecular+dynamics+with+the+AGBNP+implicit+solvent+model&amp;author=AK+Felts&amp;author=Y+Harano&amp;author=E+Gallicchio&amp;author=RM+Levy&amp;volume=56&amp;publication_year=2004&amp;pages=310-21&amp;pmid=15211514&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="citation">Feng DF, Doolittle RF. Progressive sequence alignment as a prerequisite to correct phylogenetic trees. <span><span class="ref-journal">J Mol Evol. </span>1987;<span class="ref-vol">25</span>:351–60.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3118049" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Evol&amp;title=Progressive+sequence+alignment+as+a+prerequisite+to+correct+phylogenetic+trees&amp;author=DF+Feng&amp;author=RF+Doolittle&amp;volume=25&amp;publication_year=1987&amp;pages=351-60&amp;pmid=3118049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="citation">Feng H, Vu ND, Zhou Z, Bai Y. Structural examination of &#x003c6;-value analysis in protein folding. <span><span class="ref-journal">Biochemistry. </span>2004;<span class="ref-vol">43</span>:14325–31.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15533036" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structural+examination+of+&#x003c6;-value+analysis+in+protein+folding&amp;author=H+Feng&amp;author=ND+Vu&amp;author=Z+Zhou&amp;author=Y+Bai&amp;volume=43&amp;publication_year=2004&amp;pages=14325-31&amp;pmid=15533036&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="citation">Feng H, Zhou Z, Bai Y. A protein folding pathway with multiple folding intermediates at atomic resolution. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:5026–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC555603/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15793003" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+protein+folding+pathway+with+multiple+folding+intermediates+at+atomic+resolution&amp;author=H+Feng&amp;author=Z+Zhou&amp;author=Y+Bai&amp;volume=102&amp;publication_year=2005&amp;pages=5026-31&amp;pmid=15793003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="citation">Ferguson N, Sharpe TD, Johnson CM, Fersht AR. The transition state for folding of a peripheral subunit-binding domain contains robust and ionic-strength dependent characteristics. <span><span class="ref-journal">J Mol Biol. </span>2006;<span class="ref-vol">356</span>:1237–47.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16406408" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=The+transition+state+for+folding+of+a+peripheral+subunit-binding+domain+contains+robust+and+ionic-strength+dependent+characteristics&amp;author=N+Ferguson&amp;author=TD+Sharpe&amp;author=CM+Johnson&amp;author=AR+Fersht&amp;volume=356&amp;publication_year=2006&amp;pages=1237-47&amp;pmid=16406408&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="citation">Ferguson N, Sharpe TD, Johnson CM, Schartau PJ, Fersht AR. Structural biology: analysis of &#x0201c;downhill&#x0201d; protein folding. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:14–15.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17301742" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural+biology:+analysis+of+&#x0201c;downhill&#x0201d;+protein+folding&amp;author=N+Ferguson&amp;author=TD+Sharpe&amp;author=CM+Johnson&amp;author=PJ+Schartau&amp;author=AR+Fersht&amp;volume=445&amp;publication_year=2007&amp;pages=14-15&amp;pmid=17203036&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">69. <span class="citation">Ferguson N, Sharpe TD, Schartau PJ, Sato S, Allen MD, et al. Ultra-fast barrier-limited folding in the peripheral subunit-binding domain family. <span><span class="ref-journal">J Mol Biol. </span>2005;<span class="ref-vol">353</span>:427–46.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16168437" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Ultra-fast+barrier-limited+folding+in+the+peripheral+subunit-binding+domain+family&amp;author=N+Ferguson&amp;author=TD+Sharpe&amp;author=PJ+Schartau&amp;author=S+Sato&amp;author=MD+Allen&amp;volume=353&amp;publication_year=2005&amp;pages=427-46&amp;pmid=16168437&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">70. <span class="citation">Fersht AR. Nucleation mechanisms in protein folding. <span><span class="ref-journal">Curr Opin Struct Biol. </span>1997;<span class="ref-vol">7</span>:3–9.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9032066" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Nucleation+mechanisms+in+protein+folding&amp;author=AR+Fersht&amp;volume=7&amp;publication_year=1997&amp;pages=3-9&amp;pmid=9032066&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">71. <span class="citation">Fersht AR, Sato S. &#x003a6;-value analysis and the nature of protein-folding transition states. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:7976–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC419542/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15150406" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=&#x003a6;-value+analysis+and+the+nature+of+protein-folding+transition+states&amp;author=AR+Fersht&amp;author=S+Sato&amp;volume=101&amp;publication_year=2004&amp;pages=7976-81&amp;pmid=15150406&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">72. <span class="citation">Fersht AR, Shi JP, Knill-Jones J, Lowe DM, Wilkinson AJ, et al. Hydrogen bonding and biological specificity analysed by protein engineering. <span><span class="ref-journal">Nature. </span>1985;<span class="ref-vol">314</span>:235–38.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3845322" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hydrogen+bonding+and+biological+specificity+analysed+by+protein+engineering&amp;author=AR+Fersht&amp;author=JP+Shi&amp;author=J+Knill-Jones&amp;author=DM+Lowe&amp;author=AJ+Wilkinson&amp;volume=314&amp;publication_year=1985&amp;pages=235-38&amp;pmid=3845322&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">73. <span class="citation">Fiebig KM, Dill KA. Protein core assembly processes. <span><span class="ref-journal">J Chem Phys. </span>1993;<span class="ref-vol">98</span>:3475–87.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Protein+core+assembly+processes&amp;author=KM+Fiebig&amp;author=KA+Dill&amp;volume=98&amp;publication_year=1993&amp;pages=3475-87&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">74. <span class="citation">Friel CT, Beddard GS, Radford SE. Switching two-state to three-state kinetics in the helical protein Im9 via the optimisation of stabilising non-native interactions by design. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">342</span>:261–73.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15313622" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Switching+two-state+to+three-state+kinetics+in+the+helical+protein+Im9+via+the+optimisation+of+stabilising+non-native+interactions+by+design&amp;author=CT+Friel&amp;author=GS+Beddard&amp;author=SE+Radford&amp;volume=342&amp;publication_year=2004&amp;pages=261-73&amp;pmid=15313622&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">75. <span class="citation">Garcia-Mira MM, Sadqi M, Fischer N, Sanchez-Ruiz JM, Munoz V. Experimental identification of downhill protein folding. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">298</span>:2191–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12481137" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Experimental+identification+of+downhill+protein+folding&amp;author=MM+Garcia-Mira&amp;author=M+Sadqi&amp;author=N+Fischer&amp;author=JM+Sanchez-Ruiz&amp;author=V+Munoz&amp;volume=298&amp;publication_year=2002&amp;pages=2191-95&amp;pmid=12481137&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R76">76. <span class="citation">Gellman SH. Foldamers: a manifesto. <span><span class="ref-journal">Acc Chem Res. </span>1998;<span class="ref-vol">31</span>:173–80.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acc+Chem+Res&amp;title=Foldamers:+a+manifesto&amp;author=SH+Gellman&amp;volume=31&amp;publication_year=1998&amp;pages=173-80&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R77">77. <span class="citation">Ghosh K, Ozkan SB, Dill K. The ultimate speed limit to protein folding is conformational searching. <span><span class="ref-journal">J Am Chem Soc. </span>2007;<span class="ref-vol">129</span>:11920–27.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17824609" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=The+ultimate+speed+limit+to+protein+folding+is+conformational+searching&amp;author=K+Ghosh&amp;author=SB+Ozkan&amp;author=K+Dill&amp;volume=129&amp;publication_year=2007&amp;pages=11920-27&amp;pmid=17824609&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R78">78. <span class="citation">Gianni S, Guydosh NR, Khan F, Caldas TD, Mayor U, et al. Unifying features in protein-folding mechanisms. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2003;<span class="ref-vol">100</span>:13286–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC263785/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14595026" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Unifying+features+in+protein-folding+mechanisms&amp;author=S+Gianni&amp;author=NR+Guydosh&amp;author=F+Khan&amp;author=TD+Caldas&amp;author=U+Mayor&amp;volume=100&amp;publication_year=2003&amp;pages=13286-91&amp;pmid=14595026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R79">79. <span class="citation">Gibson KD, Scheraga HA. Minimization of polypeptide energy I. Preliminary structures of bovine pancreatic ribonuclease s-peptide. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1967;<span class="ref-vol">58</span>:420–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC335651/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/5233450" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Minimization+of+polypeptide+energy+I.+Preliminary+structures+of+bovine+pancreatic+ribonuclease+s-peptide&amp;author=KD+Gibson&amp;author=HA+Scheraga&amp;volume=58&amp;publication_year=1967&amp;pages=420-27&amp;pmid=5233450&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R80">80. <span class="citation">Gilmanshin R, Williams S, Callender RH, Woodruff WH, Dyer RB. Fast events in protein folding: relaxation dynamics of secondary and tertiary structure in native apomyoglobin. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1997;<span class="ref-vol">94</span>:3709–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC20505/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9108042" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Fast+events+in+protein+folding:+relaxation+dynamics+of+secondary+and+tertiary+structure+in+native+apomyoglobin&amp;author=R+Gilmanshin&amp;author=S+Williams&amp;author=RH+Callender&amp;author=WH+Woodruff&amp;author=RB+Dyer&amp;volume=94&amp;publication_year=1997&amp;pages=3709-13&amp;pmid=9108042&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R81">81. <span class="citation">Gnanakaran S, Garcia AE. Validation of an all-atom protein force field: from dipeptides to larger peptides. <span><span class="ref-journal">J Phys Chem B. </span>2003;<span class="ref-vol">107</span>:12555–57.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Validation+of+an+all-atom+protein+force+field:+from+dipeptides+to+larger+peptides&amp;author=S+Gnanakaran&amp;author=AE+Garcia&amp;volume=107&amp;publication_year=2003&amp;pages=12555-57&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R82">82. <span class="citation">Go N, Taketomi H. Respective roles of short- and long-range interactions in protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1978;<span class="ref-vol">75</span>:559–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC411294/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/273218" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Respective+roles+of+short-+and+long-range+interactions+in+protein+folding&amp;author=N+Go&amp;author=H+Taketomi&amp;volume=75&amp;publication_year=1978&amp;pages=559-63&amp;pmid=273218&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R83">83. <span class="citation">Goldbeck RA, Thomas YG, Chen E, Esquerra RM, Kliger DS. Multiple pathways on a protein-folding energy landscape: kinetic evidence. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:2782–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC15846/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10077588" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Multiple+pathways+on+a+protein-folding+energy+landscape:+kinetic+evidence&amp;author=RA+Goldbeck&amp;author=YG+Thomas&amp;author=E+Chen&amp;author=RM+Esquerra&amp;author=DS+Kliger&amp;volume=96&amp;publication_year=1999&amp;pages=2782-87&amp;pmid=10077588&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R84">84. <span class="citation">Goldenberg DP. Genetic studies of protein stability and mechanisms of folding. <span><span class="ref-journal">Annu Rev Biophys Biomol Struct. </span>1988;<span class="ref-vol">17</span>:481–507.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3293597" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biophys+Biomol+Struct&amp;title=Genetic+studies+of+protein+stability+and+mechanisms+of+folding&amp;author=DP+Goldenberg&amp;volume=17&amp;publication_year=1988&amp;pages=481-507&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R85">85. <span class="citation">Goldenberg DP. Finding the right fold. <span><span class="ref-journal">Nat Struct Biol. </span>1999;<span class="ref-vol">6</span>:987–90.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10542081" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Finding+the+right+fold&amp;author=DP+Goldenberg&amp;volume=6&amp;publication_year=1999&amp;pages=987-90&amp;pmid=10542081&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R86">86. <span class="citation">Goodman CM, Choi S, Shandler S, DeGrado WF. Foldamers as versatile frameworks for the design and evolution of function. <span><span class="ref-journal">Nat Chem Biol. </span>2007;<span class="ref-vol">3</span>:252–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3810020/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17438550" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Chem+Biol&amp;title=Foldamers+as+versatile+frameworks+for+the+design+and+evolution+of+function&amp;author=CM+Goodman&amp;author=S+Choi&amp;author=S+Shandler&amp;author=WF+DeGrado&amp;volume=3&amp;publication_year=2007&amp;pages=252-62&amp;pmid=17438550&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R87">87. <span class="citation">Grabowski M, Joachimiak A, Otwinowski Z, Wladek M. Structural genomics: keeping up with expanding knowledge of the protein universe. <span><span class="ref-journal">Nucleic Acids Seq Topol. </span>2007;<span class="ref-vol">17</span>:347–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2885969/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17587562" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Seq+Topol&amp;title=Structural+genomics:+keeping+up+with+expanding+knowledge+of+the+protein+universe&amp;author=M+Grabowski&amp;author=A+Joachimiak&amp;author=Z+Otwinowski&amp;author=M+Wladek&amp;volume=17&amp;publication_year=2007&amp;pages=347-53&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R88">88. <span class="citation">Grantcharova V, Alm EJ, Baker D, Horwich AL. Mechanisms of protein folding. <span><span class="ref-journal">Curr Opin Struct Biol. </span>2001;<span class="ref-vol">11</span>:70–82.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11179895" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Mechanisms+of+protein+folding&amp;author=V+Grantcharova&amp;author=EJ+Alm&amp;author=D+Baker&amp;author=AL+Horwich&amp;volume=11&amp;publication_year=2001&amp;pages=70-82&amp;pmid=11179895&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R89">89. <span class="citation">Grater F, Grubmuller H. Fluctuations of primary ubiquitin folding intermediates in a force clamp. <span><span class="ref-journal">J Struct Biol. </span>2007;<span class="ref-vol">157</span>:557–69.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17306561" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Struct+Biol&amp;title=Fluctuations+of+primary+ubiquitin+folding+intermediates+in+a+force+clamp&amp;author=F+Grater&amp;author=H+Grubmuller&amp;volume=157&amp;publication_year=2007&amp;pages=557-69&amp;pmid=17306561&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R90">90. <span class="citation">Gromiha MM, Selvaraj S. Comparison between long-range interactions and contact order in determining the folding rate of two-state proteins: application of long-range order to folding rate prediction. <span><span class="ref-journal">J Mol Biol. </span>2001;<span class="ref-vol">310</span>:27–32.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11419934" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Comparison+between+long-range+interactions+and+contact+order+in+determining+the+folding+rate+of+two-state+proteins:+application+of+long-range+order+to+folding+rate+prediction&amp;author=MM+Gromiha&amp;author=S+Selvaraj&amp;volume=310&amp;publication_year=2001&amp;pages=27-32&amp;pmid=11419934&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R91">91. <span class="citation">Gromiha MM, Thangakani AM, Selvaraj S. FOLD-RATE: prediction of protein folding rates from amino acid sequence. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:W70–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1538837/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16845101" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=FOLD-RATE:+prediction+of+protein+folding+rates+from+amino+acid+sequence&amp;author=MM+Gromiha&amp;author=AM+Thangakani&amp;author=S+Selvaraj&amp;volume=34&amp;publication_year=2006&amp;pages=W70-74&amp;pmid=16845101&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R92">92. <span class="citation">Haber E, Anfinsen CB. Side-chain interactions governing the pairing of half-cystine residues in ribonuclease. <span><span class="ref-journal">J Biol Chem. </span>1962;<span class="ref-vol">237</span>:1839–44.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13903380" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Side-chain+interactions+governing+the+pairing+of+half-cystine+residues+in+ribonuclease&amp;author=E+Haber&amp;author=CB+Anfinsen&amp;volume=237&amp;publication_year=1962&amp;pages=1839-44&amp;pmid=13903380&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R93">93. <span class="citation">Hagen SJ. Probe-dependent and nonexponential relaxation kinetics: unreliable signatures of downhill protein folding. <span><span class="ref-journal">Proteins. </span>2007;<span class="ref-vol">68</span>:205–17.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17387735" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Probe-dependent+and+nonexponential+relaxation+kinetics:+unreliable+signatures+of+downhill+protein+folding&amp;author=SJ+Hagen&amp;volume=68&amp;publication_year=2007&amp;pages=205-17&amp;pmid=17387735&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R94">94. <span class="citation">Handel T, Degrado WF. Denovo design of a Zn<sup>2+</sup>-binding protein. <span><span class="ref-journal">J Am Chem Soc. </span>1990;<span class="ref-vol">112</span>:6710–11.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Denovo+design+of+a+Zn2+-binding+protein&amp;author=T+Handel&amp;author=WF+Degrado&amp;volume=112&amp;publication_year=1990&amp;pages=6710-11&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R95">95. <span class="citation">Hansmann UHE, Okamoto Y. Prediction of peptide conformation by multicanonical algorithm: new approach to the multiple-minima problem. <span><span class="ref-journal">J Comput Chem. </span>1993;<span class="ref-vol">14</span>:1333–38.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comput+Chem&amp;title=Prediction+of+peptide+conformation+by+multicanonical+algorithm:+new+approach+to+the+multiple-minima+problem&amp;author=UHE+Hansmann&amp;author=Y+Okamoto&amp;volume=14&amp;publication_year=1993&amp;pages=1333-38&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R96">96. <span class="citation">Hart WE, Istrail S. Robust proofs of NP-hardness for protein folding: general lattices and energy potentials. <span><span class="ref-journal">J Comput Biol. </span>1997;<span class="ref-vol">4</span>:1–22.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9109034" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=Robust+proofs+of+NP-hardness+for+protein+folding:+general+lattices+and+energy+potentials&amp;author=WE+Hart&amp;author=S+Istrail&amp;volume=4&amp;publication_year=1997&amp;pages=1-22&amp;pmid=9109034&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R97">97. <span class="citation">Haspel N, Tsai CJ, Wolfson H, Nussinov R. Reducing the computational complexity of protein folding via fragment folding and assembly. <span><span class="ref-journal">Protein Sci. </span>2003;<span class="ref-vol">12</span>:1177–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2323902/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12761388" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Reducing+the+computational+complexity+of+protein+folding+via+fragment+folding+and+assembly&amp;author=N+Haspel&amp;author=CJ+Tsai&amp;author=H+Wolfson&amp;author=R+Nussinov&amp;volume=12&amp;publication_year=2003&amp;pages=1177-87&amp;pmid=12761388&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R98">98. <span class="citation">Hecht MH, Das A, Go A, Bradley LH, Wei Y. De novo proteins from designed combinatorial libraries. <span><span class="ref-journal">Protein Sci. </span>2004;<span class="ref-vol">13</span>:1711–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279937/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15215517" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=De+novo+proteins+from+designed+combinatorial+libraries&amp;author=MH+Hecht&amp;author=A+Das&amp;author=A+Go&amp;author=LH+Bradley&amp;author=Y+Wei&amp;volume=13&amp;publication_year=2004&amp;pages=1711-23&amp;pmid=15215517&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R99">99. <span class="citation">Hecht MH, Richardson JS, Richardson DC, Ogden RC. De novo design, expression, and characterization of felix: a four-helix bundle protein of native-like sequence. <span><span class="ref-journal">Science. </span>1990;<span class="ref-vol">249</span>:884–91.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2392678" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=De+novo+design,+expression,+and+characterization+of+felix:+a+four-helix+bundle+protein+of+native-like+sequence&amp;author=MH+Hecht&amp;author=JS+Richardson&amp;author=DC+Richardson&amp;author=RC+Ogden&amp;volume=249&amp;publication_year=1990&amp;pages=884-91&amp;pmid=2392678&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R100">100. <span class="citation">Ho BK, Dill KA. Folding very short peptides using molecular dynamics. <span><span class="ref-journal">PLoS Comput Biol. </span>2006;<span class="ref-vol">2</span>:e27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1435986/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16617376" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=Folding+very+short+peptides+using+molecular+dynamics&amp;author=BK+Ho&amp;author=KA+Dill&amp;volume=2&amp;publication_year=2006&amp;pages=e27&amp;pmid=16617376&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R101">101. <span class="citation">Hoang L, Bedard S, Krishna MMG, Lin Y, Englander SW. Cytochrome c folding pathway: kinetic native-state hydrogen exchange. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:12173–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC129417/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12196629" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Cytochrome+c+folding+pathway:+kinetic+native-state+hydrogen+exchange&amp;author=L+Hoang&amp;author=S+Bedard&amp;author=MMG+Krishna&amp;author=Y+Lin&amp;author=SW+Englander&amp;volume=99&amp;publication_year=2002&amp;pages=12173-78&amp;pmid=12196629&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R102">102. <span class="citation">Hockenmaier J, Joshi AK, Dill KA. Routes are trees: the parsing perspective on protein folding. <span><span class="ref-journal">Proteins. </span>2006;<span class="ref-vol">66</span>:1–15.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17063473" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Routes+are+trees:+the+parsing+perspective+on+protein+folding&amp;author=J+Hockenmaier&amp;author=AK+Joshi&amp;author=KA+Dill&amp;volume=66&amp;publication_year=2006&amp;pages=1-15&amp;pmid=17063473&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R103">103. <span class="citation">Huang F, Sato S, Sharpe TD, Ying L, Fersht AR. Distinguishing between cooperative and unimodal downhill protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:123–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1765421/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17200301" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Distinguishing+between+cooperative+and+unimodal+downhill+protein+folding&amp;author=F+Huang&amp;author=S+Sato&amp;author=TD+Sharpe&amp;author=L+Ying&amp;author=AR+Fersht&amp;volume=104&amp;publication_year=2007&amp;pages=123-27&amp;pmid=17200301&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R104">104. <span class="citation">Huang JT, Cheng JP, Chen H. Secondary structure length as a determinant of folding rate of proteins with two- and three-state kinetics. <span><span class="ref-journal">Proteins. </span>2007;<span class="ref-vol">67</span>:12–17.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17206660" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Secondary+structure+length+as+a+determinant+of+folding+rate+of+proteins+with+two-+and+three-state+kinetics&amp;author=JT+Huang&amp;author=JP+Cheng&amp;author=H+Chen&amp;volume=67&amp;publication_year=2007&amp;pages=12-17&amp;pmid=17206660&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R105">105. <span class="citation">Hubner IA, Shimada J, Shakhnovich EI. &#x003a6; values and the folding transition state of protein G: utilization and interpretation of experimental data through simulation. <span><span class="ref-journal">Abstr Pap Am Chem Soc. </span>2003;<span class="ref-vol">226</span>:U450.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Abstr+Pap+Am+Chem+Soc&amp;title=&#x003a6;+values+and+the+folding+transition+state+of+protein+G:+utilization+and+interpretation+of+experimental+data+through+simulation&amp;author=IA+Hubner&amp;author=J+Shimada&amp;author=EI+Shakhnovich&amp;volume=226&amp;publication_year=2003&amp;pages=U450&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R106">106. <span class="citation">Itzhaki LS, Otzen DE, Fersht AR. The structure of the transition state for folding of chymotrypsin inhibitor 2 analysed by protein engineering methods: evidence for a nucleation-condensation mechanism for protein folding. <span><span class="ref-journal">J Mol Biol. </span>1995;<span class="ref-vol">254</span>:260–88.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7490748" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=The+structure+of+the+transition+state+for+folding+of+chymotrypsin+inhibitor+2+analysed+by+protein+engineering+methods:+evidence+for+a+nucleation-condensation+mechanism+for+protein+folding&amp;author=LS+Itzhaki&amp;author=DE+Otzen&amp;author=AR+Fersht&amp;volume=254&amp;publication_year=1995&amp;pages=260-88&amp;pmid=7490748&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R107">107. <span class="citation">Ivankov DN, Finkelstein AV. Prediction of protein folding rates from the amino acid sequence-predicted secondary structure. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:8942–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC428451/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15184682" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Prediction+of+protein+folding+rates+from+the+amino+acid+sequence-predicted+secondary+structure&amp;author=DN+Ivankov&amp;author=AV+Finkelstein&amp;volume=101&amp;publication_year=2004&amp;pages=8942-44&amp;pmid=15184682&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R108">108. <span class="citation">Jang S, Kim E, Pak Y. Direct folding simulation of &#x003b1;-helices and &#x003b2;-hairpins based on a single all-atom force field with an implicit solvation model. <span><span class="ref-journal">Proteins. </span>2007;<span class="ref-vol">66</span>:53–60.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17063490" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Direct+folding+simulation+of+&#x003b1;-helices+and+&#x003b2;-hairpins+based+on+a+single+all-atom+force+field+with+an+implicit+solvation+model&amp;author=S+Jang&amp;author=E+Kim&amp;author=Y+Pak&amp;volume=66&amp;publication_year=2007&amp;pages=53-60&amp;pmid=17063490&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R109">109. <span class="citation">Jas GS, Eaton WA, Hofrichter J. Effect of viscosity on the kinetics of &#x003b1;-helix and &#x003b2;-hairpin formation. <span><span class="ref-journal">J Phys Chem B. </span>2001;<span class="ref-vol">105</span>:261–72.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Effect+of+viscosity+on+the+kinetics+of+&#x003b1;-helix+and+&#x003b2;-hairpin+formation&amp;author=GS+Jas&amp;author=WA+Eaton&amp;author=J+Hofrichter&amp;volume=105&amp;publication_year=2001&amp;pages=261-72&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R110">110. <span class="citation">Jayachandran G, Vishal V, Pande VS. Using massively parallel simulation and Markovian models to study protein folding: examining the dynamics of the villin headpiece. <span><span class="ref-journal">J Chem Phys. </span>2006;<span class="ref-vol">124</span>:164902.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16674165" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Using+massively+parallel+simulation+and+Markovian+models+to+study+protein+folding:+examining+the+dynamics+of+the+villin+headpiece&amp;author=G+Jayachandran&amp;author=V+Vishal&amp;author=VS+Pande&amp;volume=124&amp;publication_year=2006&amp;pages=164902&amp;pmid=16674165&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R111">111. <span class="citation">Jones DT, Taylor WR, Thornton JM. A new approach to protein fold recognition. <span><span class="ref-journal">Nature. </span>1992;<span class="ref-vol">358</span>:86–89.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1614539" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+new+approach+to+protein+fold+recognition&amp;author=DT+Jones&amp;author=WR+Taylor&amp;author=JM+Thornton&amp;volume=358&amp;publication_year=1992&amp;pages=86-89&amp;pmid=1614539&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R112">112. <span class="citation">Kamtekar S, Schiffer JM, Xiong H, Babik JM, Hecht MH. Protein design by binary patterning of polar and nonpolar amino acids. <span><span class="ref-journal">Science. </span>1993;<span class="ref-vol">262</span>:1680–85.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8259512" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Protein+design+by+binary+patterning+of+polar+and+nonpolar+amino+acids&amp;author=S+Kamtekar&amp;author=JM+Schiffer&amp;author=H+Xiong&amp;author=JM+Babik&amp;author=MH+Hecht&amp;volume=262&amp;publication_year=1993&amp;pages=1680-85&amp;pmid=8259512&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R113">113. <span class="citation">Karanicolas J, Brooks CL. The origins of asymmetry in the folding transition states of protein L and protein G. <span><span class="ref-journal">Protein Sci. </span>2002;<span class="ref-vol">11</span>:2351–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2373711/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12237457" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=The+origins+of+asymmetry+in+the+folding+transition+states+of+protein+L+and+protein+G&amp;author=J+Karanicolas&amp;author=CL+Brooks&amp;volume=11&amp;publication_year=2002&amp;pages=2351-61&amp;pmid=12237457&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R114">114. <span class="citation">Karplus M, Kuriyan J. Molecular dynamics and protein function. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:6679–85.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1100762/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15870208" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Molecular+dynamics+and+protein+function&amp;author=M+Karplus&amp;author=J+Kuriyan&amp;volume=102&amp;publication_year=2005&amp;pages=6679-85&amp;pmid=15870208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R115">115. <span class="citation">Karplus M, Weaver DL. Diffusion-collision model for protein folding. <span><span class="ref-journal">Biopolymers. </span>1979;<span class="ref-vol">18</span>:1421–37.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Diffusion-collision+model+for+protein+folding&amp;author=M+Karplus&amp;author=DL+Weaver&amp;volume=18&amp;publication_year=1979&amp;pages=1421-37&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R116">116. <span class="citation">Karplus M, Weaver DL. Protein folding dynamics: the diffusion-collision model and experimental data. <span><span class="ref-journal">Protein Sci. </span>1994;<span class="ref-vol">3</span>:650–68.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2142854/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8003983" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Protein+folding+dynamics:+the+diffusion-collision+model+and+experimental+data&amp;author=M+Karplus&amp;author=DL+Weaver&amp;volume=3&amp;publication_year=1994&amp;pages=650-68&amp;pmid=8003983&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R117">117. <span class="citation">Kendrew JC. The three-dimensional structure of a protein molecule. <span><span class="ref-journal">Sci Am. </span>1961;<span class="ref-vol">205</span>:96–110.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14455128" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Am&amp;title=The+three-dimensional+structure+of+a+protein+molecule&amp;author=JC+Kendrew&amp;volume=205&amp;publication_year=1961&amp;pages=96-110&amp;pmid=14455128&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R118">118. <span class="citation">Kim DE, Gu H, Baker D. The sequences of small proteins are not extensively optimized for rapid folding by natural selection. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1998;<span class="ref-vol">95</span>:4982–86.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC20199/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9560214" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+sequences+of+small+proteins+are+not+extensively+optimized+for+rapid+folding+by+natural+selection&amp;author=DE+Kim&amp;author=H+Gu&amp;author=D+Baker&amp;volume=95&amp;publication_year=1998&amp;pages=4982-86&amp;pmid=9560214&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R119">119. <span class="citation">Kim PS, Baldwin RL. Specific intermediates in the folding reactions of small proteins and the mechanism of protein folding. <span><span class="ref-journal">Annu Rev Biochem. </span>1982;<span class="ref-vol">51</span>:459–89.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6287919" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biochem&amp;title=Specific+intermediates+in+the+folding+reactions+of+small+proteins+and+the+mechanism+of+protein+folding&amp;author=PS+Kim&amp;author=RL+Baldwin&amp;volume=51&amp;publication_year=1982&amp;pages=459-89&amp;pmid=6287919&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R120">120. <span class="citation">Kirshenbaum K, Zuckermann RN, Dill KA. Designing polymers that mimic biomolecules. <span><span class="ref-journal">Curr Opin Struct Biol. </span>1999;<span class="ref-vol">9</span>:530–35.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10449369" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Designing+polymers+that+mimic+biomolecules&amp;author=K+Kirshenbaum&amp;author=RN+Zuckermann&amp;author=KA+Dill&amp;volume=9&amp;publication_year=1999&amp;pages=530-35&amp;pmid=10449369&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R121">121. <span class="citation">Klepeis JL, Floudas CA. ASTRO-FOLD: a combinatorial and global optimization framework for ab initio prediction of three-dimensional structures of proteins from the amino acid sequence. <span><span class="ref-journal">Biophys J. </span>2003;<span class="ref-vol">85</span>:2119–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1303441/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14507680" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=ASTRO-FOLD:+a+combinatorial+and+global+optimization+framework+for+ab+initio+prediction+of+three-dimensional+structures+of+proteins+from+the+amino+acid+sequence&amp;author=JL+Klepeis&amp;author=CA+Floudas&amp;volume=85&amp;publication_year=2003&amp;pages=2119-46&amp;pmid=14507680&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R122">122. <span class="citation">Knott M, Chan HS. Criteria for downhill protein folding: calorimetry, chevron plot, kinetic relaxation, and single-molecule radius of gyration in chain models with subdued degrees of cooperativity. <span><span class="ref-journal">Proteins. </span>2006;<span class="ref-vol">65</span>:373–91.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16909416" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Criteria+for+downhill+protein+folding:+calorimetry,+chevron+plot,+kinetic+relaxation,+and+single-molecule+radius+of+gyration+in+chain+models+with+subdued+degrees+of+cooperativity&amp;author=M+Knott&amp;author=HS+Chan&amp;volume=65&amp;publication_year=2006&amp;pages=373-91&amp;pmid=16909416&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R123">123. <span class="citation">Korzhnev DM, Salvatella X, Vendruscolo M, Di Nardo AA, Davidson AR, et al. Low-populated folding intermediates of Fyn SH3 characterized by relaxation dispersion NMR. <span><span class="ref-journal">Nature. </span>2004;<span class="ref-vol">430</span>:586–90.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15282609" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Low-populated+folding+intermediates+of+Fyn+SH3+characterized+by+relaxation+dispersion+NMR&amp;author=DM+Korzhnev&amp;author=X+Salvatella&amp;author=M+Vendruscolo&amp;author=AA+Di+Nardo&amp;author=AR+Davidson&amp;volume=430&amp;publication_year=2004&amp;pages=586-90&amp;pmid=15282609&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R124">124. <span class="citation">Krieger F, Fierz B, Bieri O, Drewello M, Kiefhaber T. Dynamics of unfolded polypeptide chains as model for the earliest steps in protein folding. <span><span class="ref-journal">J Mol Biol. </span>2003;<span class="ref-vol">332</span>:265–74.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12946363" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Dynamics+of+unfolded+polypeptide+chains+as+model+for+the+earliest+steps+in+protein+folding&amp;author=F+Krieger&amp;author=B+Fierz&amp;author=O+Bieri&amp;author=M+Drewello&amp;author=T+Kiefhaber&amp;volume=332&amp;publication_year=2003&amp;pages=265-74&amp;pmid=12946363&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R125">125. <span class="citation">Krishna MM, Hoang L, Lin Y, Englander SW. Hydrogen exchange methods to study protein folding. <span><span class="ref-journal">Methods. </span>2004;<span class="ref-vol">34</span>:51–64.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15283915" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Hydrogen+exchange+methods+to+study+protein+folding&amp;author=MM+Krishna&amp;author=L+Hoang&amp;author=Y+Lin&amp;author=SW+Englander&amp;volume=34&amp;publication_year=2004&amp;pages=51-64&amp;pmid=15283915&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R126">126. <span class="citation">Krishna MMG, Maity H, Rumbley JN, Lin Y, Englander SW. Order of steps in the cytochrome <em>c</em> folding pathway: evidence for a sequential stabilization mechanism. <span><span class="ref-journal">J Mol Biol. </span>2006;<span class="ref-vol">359</span>:1411–20.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16690080" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Order+of+steps+in+the+cytochrome+c+folding+pathway:+evidence+for+a+sequential+stabilization+mechanism&amp;author=MMG+Krishna&amp;author=H+Maity&amp;author=JN+Rumbley&amp;author=Y+Lin&amp;author=SW+Englander&amp;volume=359&amp;publication_year=2006&amp;pages=1411-20&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R127">127. <span class="citation">Kubelka J, Chiu TK, Davies DR, Eaton WA, Hofrichter J. Sub-microsecond protein folding. <span><span class="ref-journal">J Mol Biol. </span>2006;<span class="ref-vol">359</span>:546–53.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16643946" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Sub-microsecond+protein+folding&amp;author=J+Kubelka&amp;author=TK+Chiu&amp;author=DR+Davies&amp;author=WA+Eaton&amp;author=J+Hofrichter&amp;volume=359&amp;publication_year=2006&amp;pages=546-53&amp;pmid=16643946&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R128">128. <span class="citation">Kubelka J, Hofrichter J, Eaton WA. The protein folding &#x02018;speed limit&#x02019; <span><span class="ref-journal">Curr Opin Struct Biol. </span>2004;<span class="ref-vol">14</span>:76–88.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15102453" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=The+protein+folding+&#x02018;speed+limit&#x02019;&amp;author=J+Kubelka&amp;author=J+Hofrichter&amp;author=WA+Eaton&amp;volume=14&amp;publication_year=2004&amp;pages=76-88&amp;pmid=15102453&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R129">129. <span class="citation">Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, Baker D. Design of a novel globular protein fold with atomic-level accuracy. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">302</span>:1364–68.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14631033" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Design+of+a+novel+globular+protein+fold+with+atomic-level+accuracy&amp;author=B+Kuhlman&amp;author=G+Dantas&amp;author=GC+Ireton&amp;author=G+Varani&amp;author=BL+Stoddard&amp;volume=302&amp;publication_year=2003&amp;pages=1364-68&amp;pmid=14631033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R130">130. <span class="citation">Kumar S, Bouzida D, Swendsen RH, Kollman PA, Rosenberg JM. The weighted histogram analysis method for free-energy calculations on biomolecules. 1. The method. <span><span class="ref-journal">J Comput Chem. </span>1992;<span class="ref-vol">13</span>:1011–21.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comput+Chem&amp;title=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules.+1.+The+method&amp;author=S+Kumar&amp;author=D+Bouzida&amp;author=RH+Swendsen&amp;author=PA+Kollman&amp;author=JM+Rosenberg&amp;volume=13&amp;publication_year=1992&amp;pages=1011-21&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R131">131. <span class="citation">Kuznetsov IB, Rackovsky S. Class-specific correlations between protein folding rate, structure-derived, and sequence-derived descriptors. <span><span class="ref-journal">Proteins. </span>2004;<span class="ref-vol">54</span>:333–41.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14696195" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Class-specific+correlations+between+protein+folding+rate,+structure-derived,+and+sequence-derived+descriptors&amp;author=IB+Kuznetsov&amp;author=S+Rackovsky&amp;volume=54&amp;publication_year=2004&amp;pages=333-41&amp;pmid=14696195&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R132">132. <span class="citation">Lau KF, Dill KA. A lattice statistical mechanics model of the conformational and sequence spaces of proteins. <span><span class="ref-journal">Macromolecules. </span>1989;<span class="ref-vol">22</span>:3986–97.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Macromolecules&amp;title=A+lattice+statistical+mechanics+model+of+the+conformational+and+sequence+spaces+of+proteins&amp;author=KF+Lau&amp;author=KA+Dill&amp;volume=22&amp;publication_year=1989&amp;pages=3986-97&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R133">133. <span class="citation">Laurence TA, Kong X, J&#x000e4;ger M, Weiss S. Probing structural heterogeneities and fluctuations of nucleic acids and denatured proteins. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:17348–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1297681/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16287971" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Probing+structural+heterogeneities+and+fluctuations+of+nucleic+acids+and+denatured+proteins&amp;author=TA+Laurence&amp;author=X+Kong&amp;author=M+J&#x000e4;ger&amp;author=S+Weiss&amp;volume=102&amp;publication_year=2005&amp;pages=17348-53&amp;pmid=16287971&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R134">134. <span class="citation">Lee BC, Zuckermann RN, Dill KA. Folding a nonbiological polymer into a compact multihelical structure. <span><span class="ref-journal">J Am Chem Soc. </span>2005;<span class="ref-vol">127</span>:10999–1009.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16076207" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Folding+a+nonbiological+polymer+into+a+compact+multihelical+structure&amp;author=BC+Lee&amp;author=RN+Zuckermann&amp;author=KA+Dill&amp;volume=127&amp;publication_year=2005&amp;pages=10999-1009&amp;pmid=16076207&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R135">135. <span class="citation">Lei H, Duan Y. Ab initio folding of albumin binding domain from all-atom molecular dynamics simulation. <span><span class="ref-journal">J Phys Chem B. </span>2007;<span class="ref-vol">111</span>:5458–63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17458992" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Ab+initio+folding+of+albumin+binding+domain+from+all-atom+molecular+dynamics+simulation&amp;author=H+Lei&amp;author=Y+Duan&amp;volume=111&amp;publication_year=2007&amp;pages=5458-63&amp;pmid=17458992&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R136">136. <span class="citation">Lei H, Duan Y. Two-stage folding of Hp-35 from ab initio simulations. <span><span class="ref-journal">J Mol Biol. </span>2007;<span class="ref-vol">370</span>:196–206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701201/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17512537" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Two-stage+folding+of+Hp-35+from+ab+initio+simulations&amp;author=H+Lei&amp;author=Y+Duan&amp;volume=370&amp;publication_year=2007&amp;pages=196-206&amp;pmid=17512537&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R137">137. <span class="citation">Lei H, Wu C, Liu H, Duan Y. Folding free-energy landscape of villin headpiece subdomain from molecular dynamics simulations. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:4925–30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1829241/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17360390" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Folding+free-energy+landscape+of+villin+headpiece+subdomain+from+molecular+dynamics+simulations&amp;author=H+Lei&amp;author=C+Wu&amp;author=H+Liu&amp;author=Y+Duan&amp;volume=104&amp;publication_year=2007&amp;pages=4925-30&amp;pmid=17360390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R138">138. <span class="citation">Leopold PE, Montal M, Onuchic JN. Protein folding funnels: a kinetic approach to the sequence-structure relationship. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1992;<span class="ref-vol">89</span>:8721–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC49992/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1528885" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Protein+folding+funnels:+a+kinetic+approach+to+the+sequence-structure+relationship&amp;author=PE+Leopold&amp;author=M+Montal&amp;author=JN+Onuchic&amp;volume=89&amp;publication_year=1992&amp;pages=8721-25&amp;pmid=1528885&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R139">139. <span class="citation">Lesk AM, Rose GD. Folding units in globular proteins. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1981;<span class="ref-vol">78</span>:4304–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC319778/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6945585" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Folding+units+in+globular+proteins&amp;author=AM+Lesk&amp;author=GD+Rose&amp;volume=78&amp;publication_year=1981&amp;pages=4304-8&amp;pmid=6945585&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R140">140. <span class="citation">Levitt M. Molecular dynamics of native protein. I. Computer simulation of trajectories. <span><span class="ref-journal">J Mol Biol. </span>1983;<span class="ref-vol">168</span>:595–617.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6193280" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Molecular+dynamics+of+native+protein.+I.+Computer+simulation+of+trajectories&amp;author=M+Levitt&amp;volume=168&amp;publication_year=1983&amp;pages=595-617&amp;pmid=6193280&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R141">141. <span class="citation">Li Z, Scheraga HA. Monte Carlo&#x02013;minimization approach to the multiple-minima problem in protein folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1987;<span class="ref-vol">84</span>:6611–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC299132/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3477791" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Monte+Carlo&#x02013;minimization+approach+to+the+multiple-minima+problem+in+protein+folding&amp;author=Z+Li&amp;author=HA+Scheraga&amp;volume=84&amp;publication_year=1987&amp;pages=6611-15&amp;pmid=3477791&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R142">142. <span class="citation">Lindberg MO, Haglund E, Hubner IA, Shakhnovich EI, Oliveberg M. Identification of the minimal protein-folding nucleus through loop-entropy perturbations. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:4083–88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1449650/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16505376" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Identification+of+the+minimal+protein-folding+nucleus+through+loop-entropy+perturbations&amp;author=MO+Lindberg&amp;author=E+Haglund&amp;author=IA+Hubner&amp;author=EI+Shakhnovich&amp;author=M+Oliveberg&amp;volume=103&amp;publication_year=2006&amp;pages=4083-88&amp;pmid=16505376&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R143">143. <span class="citation">Lindberg MO, Oliveberg M. Malleability of protein folding pathways: a simple reason for complex behaviour. <span><span class="ref-journal">Curr Opin Struct Biol. </span>2007;<span class="ref-vol">17</span>:21–29.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17251003" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Malleability+of+protein+folding+pathways:+a+simple+reason+for+complex+behaviour&amp;author=MO+Lindberg&amp;author=M+Oliveberg&amp;volume=17&amp;publication_year=2007&amp;pages=21-29&amp;pmid=17251003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R144">144. <span class="citation">Lindorff-Larsen K, Paci E, Serrano L, Dobson CM, Vendruscolo M. Calculation of mutational free energy changes in transition states for protein folding. <span><span class="ref-journal">Biophys J. </span>2003;<span class="ref-vol">85</span>:1207–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1303238/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12885664" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=Calculation+of+mutational+free+energy+changes+in+transition+states+for+protein+folding&amp;author=K+Lindorff-Larsen&amp;author=E+Paci&amp;author=L+Serrano&amp;author=CM+Dobson&amp;author=M+Vendruscolo&amp;volume=85&amp;publication_year=2003&amp;pages=1207-14&amp;pmid=12885664&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R145">145. <span class="citation">Looger LL, Dwyer MA, Smith JJ, Hellinga HW. Computational design of receptor and sensor proteins with novel functions. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">423</span>:185–90.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12736688" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Computational+design+of+receptor+and+sensor+proteins+with+novel+functions&amp;author=LL+Looger&amp;author=MA+Dwyer&amp;author=JJ+Smith&amp;author=HW+Hellinga&amp;volume=423&amp;publication_year=2003&amp;pages=185-90&amp;pmid=12736688&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R146">146. <span class="citation">Lucas A, Huang L, Joshi A, Dill KA. Statistical mechanics of helix bundles using a dynamic programming approach. <span><span class="ref-journal">J Am Chem Soc. </span>2007;<span class="ref-vol">129</span>:4272–81.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17362002" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Statistical+mechanics+of+helix+bundles+using+a+dynamic+programming+approach&amp;author=A+Lucas&amp;author=L+Huang&amp;author=A+Joshi&amp;author=KA+Dill&amp;volume=129&amp;publication_year=2007&amp;pages=4272-81&amp;pmid=17362002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R147">147. <span class="citation">Lucent D, Vishal V, Pande VS. Protein folding under confinement: a role for solvent. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:10430–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1965530/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17563390" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Protein+folding+under+confinement:+a+role+for+solvent&amp;author=D+Lucent&amp;author=V+Vishal&amp;author=VS+Pande&amp;volume=104&amp;publication_year=2007&amp;pages=10430-34&amp;pmid=17563390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R148">148. <span class="citation">Ma H, Gruebele M. Low barrier kinetics: dependence on observables and free energy surface. <span><span class="ref-journal">J Comput Chem. </span>2006;<span class="ref-vol">27</span>:125–34.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16302178" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comput+Chem&amp;title=Low+barrier+kinetics:+dependence+on+observables+and+free+energy+surface&amp;author=H+Ma&amp;author=M+Gruebele&amp;volume=27&amp;publication_year=2006&amp;pages=125-34&amp;pmid=16302178&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R149">149. <span class="citation">Maity H, Maity M, Krishna MM, Mayne L, Englander SW. Protein folding: the stepwise assembly of foldon units. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:4741–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC555724/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15774579" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Protein+folding:+the+stepwise+assembly+of+foldon+units&amp;author=H+Maity&amp;author=M+Maity&amp;author=MM+Krishna&amp;author=L+Mayne&amp;author=SW+Englander&amp;volume=102&amp;publication_year=2005&amp;pages=4741-46&amp;pmid=15774579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R150">150. <span class="citation">Makarov DE, Keller CA, Plaxco KW, Metiu H. How the folding rate constant of simple, single-domain proteins depends on the number of native contacts. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:3535–39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122558/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11904417" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=How+the+folding+rate+constant+of+simple,+single-domain+proteins+depends+on+the+number+of+native+contacts&amp;author=DE+Makarov&amp;author=CA+Keller&amp;author=KW+Plaxco&amp;author=H+Metiu&amp;volume=99&amp;publication_year=2002&amp;pages=3535-39&amp;pmid=11904417&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R151">151. <span class="citation">Matagne A, Radford SE, Dobson CM. Fast and slow tracks in lysozyme folding: insight into the role of domains in the folding process. <span><span class="ref-journal">J Mol Biol. </span>1997;<span class="ref-vol">267</span>:1068–74.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9150396" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Fast+and+slow+tracks+in+lysozyme+folding:+insight+into+the+role+of+domains+in+the+folding+process&amp;author=A+Matagne&amp;author=SE+Radford&amp;author=CM+Dobson&amp;volume=267&amp;publication_year=1997&amp;pages=1068-74&amp;pmid=9150396&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R152">152. <span class="citation">Matheson RR, Jr, Scheraga HA. A method for predicting nucleation sites for protein folding based on hydrophobic contacts. <span><span class="ref-journal">Macromolecules. </span>1978;<span class="ref-vol">11</span>:819–29.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Macromolecules&amp;title=A+method+for+predicting+nucleation+sites+for+protein+folding+based+on+hydrophobic+contacts&amp;author=RR+Matheson&amp;author=HA+Scheraga&amp;volume=11&amp;publication_year=1978&amp;pages=819-29&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R153">153. <span class="citation">Matouschek A, Kellis JT, Jr, Serrano L, Fersht AR. Mapping the transition state and pathway of protein folding by protein engineering. <span><span class="ref-journal">Nature. </span>1989;<span class="ref-vol">340</span>:122–26.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2739734" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mapping+the+transition+state+and+pathway+of+protein+folding+by+protein+engineering&amp;author=A+Matouschek&amp;author=JT+Kellis&amp;author=L+Serrano&amp;author=AR+Fersht&amp;volume=340&amp;publication_year=1989&amp;pages=122-26&amp;pmid=2739734&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R154">154. <span class="citation">Maxwell KL, Wildes D, Zarrine-Afsar A, De Los Rios MA, Brown AG, et al. Protein folding: defining a &#x0201c;standard&#x0201d; set of experimental conditions and a preliminary kinetic data set of two-state proteins. <span><span class="ref-journal">Protein Sci. </span>2005;<span class="ref-vol">14</span>:602–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279278/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15689503" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Protein+folding:+defining+a+&#x0201c;standard&#x0201d;+set+of+experimental+conditions+and+a+preliminary+kinetic+data+set+of+two-state+proteins&amp;author=KL+Maxwell&amp;author=D+Wildes&amp;author=A+Zarrine-Afsar&amp;author=MA+De+Los+Rios&amp;author=AG+Brown&amp;volume=14&amp;publication_year=2005&amp;pages=602-16&amp;pmid=15689503&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R155">155. <span class="citation">McCammon JA, Gelin BR, Karplus M. Dynamics of folded proteins. <span><span class="ref-journal">Nature. </span>1977;<span class="ref-vol">267</span>:585–90.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/301613" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dynamics+of+folded+proteins&amp;author=JA+McCammon&amp;author=BR+Gelin&amp;author=M+Karplus&amp;volume=267&amp;publication_year=1977&amp;pages=585-90&amp;pmid=301613&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R156">156. <span class="citation">Meisner WK, Sosnick TR. Barrier-limited, microsecond folding of a stable protein measured with hydrogen exchange: implications for downhill folding. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:15639–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC524831/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15505204" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Barrier-limited,+microsecond+folding+of+a+stable+protein+measured+with+hydrogen+exchange:+implications+for+downhill+folding&amp;author=WK+Meisner&amp;author=TR+Sosnick&amp;volume=101&amp;publication_year=2004&amp;pages=15639-44&amp;pmid=15505204&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R157">157. <span class="citation">Mello CC, Barrick D. An experimentally determined protein folding energy landscape. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:14102–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC521126/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15377792" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=An+experimentally+determined+protein+folding+energy+landscape&amp;author=CC+Mello&amp;author=D+Barrick&amp;volume=101&amp;publication_year=2004&amp;pages=14102-7&amp;pmid=15377792&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R158">158. <span class="citation">Mezei M. Chameleon sequences in the PDB. <span><span class="ref-journal">Protein Eng. </span>1998;<span class="ref-vol">11</span>:411–14.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9725618" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Eng&amp;title=Chameleon+sequences+in+the+PDB&amp;author=M+Mezei&amp;volume=11&amp;publication_year=1998&amp;pages=411-14&amp;pmid=9725618&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R159">159. <span class="citation">Micheletti C, Banavar JR, Maritan A, Seno F. Protein structures and optimal folding from a geometrical variational principle. <span><span class="ref-journal">Phys Rev Lett. </span>1999;<span class="ref-vol">82</span>:3372–75.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys+Rev+Lett&amp;title=Protein+structures+and+optimal+folding+from+a+geometrical+variational+principle&amp;author=C+Micheletti&amp;author=JR+Banavar&amp;author=A+Maritan&amp;author=F+Seno&amp;volume=82&amp;publication_year=1999&amp;pages=3372-75&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R160">160. <span class="citation">Miller EJ, Fischer KF, Marqusee S. Experimental evaluation of topological parameters determining protein-folding rates. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:10359–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC124919/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12149462" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Experimental+evaluation+of+topological+parameters+determining+protein-folding+rates&amp;author=EJ+Miller&amp;author=KF+Fischer&amp;author=S+Marqusee&amp;volume=99&amp;publication_year=2002&amp;pages=10359-63&amp;pmid=12149462&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R161">161. <span class="citation">Miller R, Danko CA, Fasolka MJ, Balazs AC, Chan HS, Dill KA. Folding kinetics of proteins and copolymers. <span><span class="ref-journal">J Chem Phys. </span>1992;<span class="ref-vol">96</span>:768–80.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Folding+kinetics+of+proteins+and+copolymers&amp;author=R+Miller&amp;author=CA+Danko&amp;author=MJ+Fasolka&amp;author=AC+Balazs&amp;author=HS+Chan&amp;volume=96&amp;publication_year=1992&amp;pages=768-80&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R162">162. <span class="citation">Minor DL, Kim PS. Context-dependent secondary structure formation of a designed protein sequence. <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">380</span>:730–34.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8614471" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Context-dependent+secondary+structure+formation+of+a+designed+protein+sequence&amp;author=DL+Minor&amp;author=PS+Kim&amp;volume=380&amp;publication_year=1996&amp;pages=730-34&amp;pmid=8614471&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R163">163. <span class="citation">Mirny L, Shakhnovich E. Protein folding theory: from lattice to all-atom models. <span><span class="ref-journal">Annu Rev Biophys Biomol Struct. </span>2001;<span class="ref-vol">30</span>:361–96.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11340064" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biophys+Biomol+Struct&amp;title=Protein+folding+theory:+from+lattice+to+all-atom+models&amp;author=L+Mirny&amp;author=E+Shakhnovich&amp;volume=30&amp;publication_year=2001&amp;pages=361-96&amp;pmid=11340064&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R164">164. <span class="citation">Moult J. A decade of CASP: progress, bottlenecks and prognosis in protein structure prediction. <span><span class="ref-journal">Curr Opin Struct Biol. </span>2005;<span class="ref-vol">15</span>:285–89.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15939584" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=A+decade+of+CASP:+progress,+bottlenecks+and+prognosis+in+protein+structure+prediction&amp;author=J+Moult&amp;volume=15&amp;publication_year=2005&amp;pages=285-89&amp;pmid=15939584&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R165">165. <span class="citation">Moult J, Pedersen JT, Judson R, Fidelis K. A large-scale experiment to assess protein structure prediction methods. <span><span class="ref-journal">Proteins. </span>1995;<span class="ref-vol">23</span>:ii–iv.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8710822" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A+large-scale+experiment+to+assess+protein+structure+prediction+methods&amp;author=J+Moult&amp;author=JT+Pedersen&amp;author=R+Judson&amp;author=K+Fidelis&amp;volume=23&amp;publication_year=1995&amp;pages=ii-iv&amp;pmid=8710822&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R166">166. <span class="citation">Mukhopadhyay S, Krishnan R, Lemke EA, Lindquist S, Deniz AA. A natively unfolded yeast prion monomer adopts an ensemble of collapsed and rapidly fluctuating structures. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:2649–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1815236/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17299036" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+natively+unfolded+yeast+prion+monomer+adopts+an+ensemble+of+collapsed+and+rapidly+fluctuating+structures&amp;author=S+Mukhopadhyay&amp;author=R+Krishnan&amp;author=EA+Lemke&amp;author=S+Lindquist&amp;author=AA+Deniz&amp;volume=104&amp;publication_year=2007&amp;pages=2649-54&amp;pmid=17299036&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R167">167. <span class="citation">Munoz V. Thermodynamics and kinetics of downhill protein folding investigated with a simple statistical mechanical model. <span><span class="ref-journal">Int J Quant Chem. </span>2002;<span class="ref-vol">90</span>:1522–28.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Quant+Chem&amp;title=Thermodynamics+and+kinetics+of+downhill+protein+folding+investigated+with+a+simple+statistical+mechanical+model&amp;author=V+Munoz&amp;volume=90&amp;publication_year=2002&amp;pages=1522-28&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R168">168. <span class="citation">Munoz V, Sanchez-Ruiz JM. Exploring protein-folding ensembles: a variable-barrier model for the analysis of equilibrium unfolding experiments. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:17646–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC539728/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15591110" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Exploring+protein-folding+ensembles:+a+variable-barrier+model+for+the+analysis+of+equilibrium+unfolding+experiments&amp;author=V+Munoz&amp;author=JM+Sanchez-Ruiz&amp;volume=101&amp;publication_year=2004&amp;pages=17646-51&amp;pmid=15591110&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R169">169. <span class="citation">Myers JK, Oas TG. Preorganized secondary structure as an important determinant of fast protein folding. <span><span class="ref-journal">Nat Struct Biol. </span>2001;<span class="ref-vol">8</span>:552–58.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11373626" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Preorganized+secondary+structure+as+an+important+determinant+of+fast+protein+folding&amp;author=JK+Myers&amp;author=TG+Oas&amp;volume=8&amp;publication_year=2001&amp;pages=552-58&amp;pmid=11373626&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R170">170. <span class="citation">Naganathan AN, Perez-Jimenez R, Sanchez-Ruiz JM, Munoz V. Robustness of downhill folding: guidelines for the analysis of equilibrium folding experiments on small proteins. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:7435–49.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15895987" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Robustness+of+downhill+folding:+guidelines+for+the+analysis+of+equilibrium+folding+experiments+on+small+proteins&amp;author=AN+Naganathan&amp;author=R+Perez-Jimenez&amp;author=JM+Sanchez-Ruiz&amp;author=V+Munoz&amp;volume=44&amp;publication_year=2005&amp;pages=7435-49&amp;pmid=15895987&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R171">171. <span class="citation">Nemethy G, Scheraga HA. Theoretical determination of sterically allowed conformations of a polypeptide chain by a computer method. <span><span class="ref-journal">Biopolymers. </span>1965;<span class="ref-vol">3</span>:155–84.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Theoretical+determination+of+sterically+allowed+conformations+of+a+polypeptide+chain+by+a+computer+method&amp;author=G+Nemethy&amp;author=HA+Scheraga&amp;volume=3&amp;publication_year=1965&amp;pages=155-84&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R172">172. <span class="citation">No&#x000e9; F, Horenko I, Sch&#x000fc;tte C, Smith JC. Hierarchical analysis of conformational dynamics in biomolecules: transition networks of metastable states. <span><span class="ref-journal">J Chem Phys. </span>2007;<span class="ref-vol">126</span>:155102.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17461666" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Hierarchical+analysis+of+conformational+dynamics+in+biomolecules:+transition+networks+of+metastable+states&amp;author=F+No&#x000e9;&amp;author=I+Horenko&amp;author=C+Sch&#x000fc;tte&amp;author=JC+Smith&amp;volume=126&amp;publication_year=2007&amp;pages=155102&amp;pmid=17461666&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R173">173. <span class="citation">O&#x02019;Neil KT, Hoess RH, DeGrado WF. Design of DNA-binding peptides based on the leucine zipper motif. <span><span class="ref-journal">Science. </span>1990;<span class="ref-vol">249</span>:774–78.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2389143" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Design+of+DNA-binding+peptides+based+on+the+leucine+zipper+motif&amp;author=KT+O&#x02019;Neil&amp;author=RH+Hoess&amp;author=WF+DeGrado&amp;volume=249&amp;publication_year=1990&amp;pages=774-78&amp;pmid=2389143&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R174">174. <span class="citation">Oldziej S, Czaplewski C, Liwo A, Chinchio M, Nanias M, et al. Physics-based protein-structure prediction using a hierarchical protocol based on the UNRES force field: assessment in two blind tests. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:7547–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1140449/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15894609" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Physics-based+protein-structure+prediction+using+a+hierarchical+protocol+based+on+the+UNRES+force+field:+assessment+in+two+blind+tests&amp;author=S+Oldziej&amp;author=C+Czaplewski&amp;author=A+Liwo&amp;author=M+Chinchio&amp;author=M+Nanias&amp;volume=102&amp;publication_year=2005&amp;pages=7547-52&amp;pmid=15894609&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R175">175. <span class="citation">Onuchic JN, Wolynes PG. Energy landscapes, glass transitions, and chemical-reaction dynamics in biomolecular or solvent environment. <span><span class="ref-journal">J Chem Phys. </span>1993;<span class="ref-vol">98</span>:2218–24.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Energy+landscapes,+glass+transitions,+and+chemical-reaction+dynamics+in+biomolecular+or+solvent+environment&amp;author=JN+Onuchic&amp;author=PG+Wolynes&amp;volume=98&amp;publication_year=1993&amp;pages=2218-24&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R176">176. <span class="citation">Ozkan SB, Bahar I, Dill KA. Transition states and the meaning of &#x003c6;-values in protein folding kinetics. <span><span class="ref-journal">Nat Struct Biol. </span>2001;<span class="ref-vol">8</span>:765–69.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11524678" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Transition+states+and+the+meaning+of+&#x003c6;-values+in+protein+folding+kinetics&amp;author=SB+Ozkan&amp;author=I+Bahar&amp;author=KA+Dill&amp;volume=8&amp;publication_year=2001&amp;pages=765-69&amp;pmid=11524678&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R177">177. <span class="citation">Ozkan SB, Wu GHA, Chodera JD, Dill KA. Protein folding by zipping and assembly. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:11987–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1924571/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17620603" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Protein+folding+by+zipping+and+assembly&amp;author=SB+Ozkan&amp;author=GHA+Wu&amp;author=JD+Chodera&amp;author=KA+Dill&amp;volume=104&amp;publication_year=2007&amp;pages=11987-92&amp;pmid=17620603&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R178">178. <span class="citation">Paci E, Vendruscolo M, Dobson CM, Karplus M. Determination of a transition state at atomic resolution from protein engineering data. <span><span class="ref-journal">J Mol Biol. </span>2002;<span class="ref-vol">324</span>:151–63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12421565" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Determination+of+a+transition+state+at+atomic+resolution+from+protein+engineering+data&amp;author=E+Paci&amp;author=M+Vendruscolo&amp;author=CM+Dobson&amp;author=M+Karplus&amp;volume=324&amp;publication_year=2002&amp;pages=151-63&amp;pmid=12421565&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R179">179. <span class="citation">Patch JA, Barron AE. Helical peptoid mimics of magainin-2 amide. <span><span class="ref-journal">J Am Chem Soc. </span>2003;<span class="ref-vol">125</span>:12092–93.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14518985" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Helical+peptoid+mimics+of+magainin-2+amide&amp;author=JA+Patch&amp;author=AE+Barron&amp;volume=125&amp;publication_year=2003&amp;pages=12092-93&amp;pmid=14518985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R180">180. <span class="citation">Pauling L, Corey RB. Atomic coordinates and structure factors for two helical configurations of polypeptide chains. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1951;<span class="ref-vol">37</span>:235–40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1063348/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14834145" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Atomic+coordinates+and+structure+factors+for+two+helical+configurations+of+polypeptide+chains&amp;author=L+Pauling&amp;author=RB+Corey&amp;volume=37&amp;publication_year=1951&amp;pages=235-40&amp;pmid=14834145&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R181">181. <span class="citation">Pauling L, Corey RB, Branson HR. The structure of proteins: two hydrogen-bonded helical configurations of the polypeptide chain. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1951;<span class="ref-vol">37</span>:205–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1063337/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14816373" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+structure+of+proteins:+two+hydrogen-bonded+helical+configurations+of+the+polypeptide+chain&amp;author=L+Pauling&amp;author=RB+Corey&amp;author=HR+Branson&amp;volume=37&amp;publication_year=1951&amp;pages=205-11&amp;pmid=14816373&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R182">182. <span class="citation">Pieper U, Eswar N, Braberg H, Madhusudhan MS, Davis FP, et al. MODBASE, a database of annotated comparative protein structure models, and associated resources. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:D217–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC308829/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14681398" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MODBASE,+a+database+of+annotated+comparative+protein+structure+models,+and+associated+resources&amp;author=U+Pieper&amp;author=N+Eswar&amp;author=H+Braberg&amp;author=MS+Madhusudhan&amp;author=FP+Davis&amp;volume=32&amp;publication_year=2004&amp;pages=D217-22&amp;pmid=14681398&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R183">183. <span class="citation">Pitera JW, Swope W. Understanding folding and design: replica-exchange simulations of &#x0201c;Trp-cage&#x0201d; miniproteins. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2003;<span class="ref-vol">100</span>:7587–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC164630/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12808142" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Understanding+folding+and+design:+replica-exchange+simulations+of+&#x0201c;Trp-cage&#x0201d;+miniproteins&amp;author=JW+Pitera&amp;author=W+Swope&amp;volume=100&amp;publication_year=2003&amp;pages=7587-92&amp;pmid=12808142&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R184">184. <span class="citation">Plaxco KW, Simons KT, Baker D. Contact order, transition state placement and the refolding rates of single domain proteins. <span><span class="ref-journal">J Mol Biol. </span>1998;<span class="ref-vol">277</span>:985–94.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9545386" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Contact+order,+transition+state+placement+and+the+refolding+rates+of+single+domain+proteins&amp;author=KW+Plaxco&amp;author=KT+Simons&amp;author=D+Baker&amp;volume=277&amp;publication_year=1998&amp;pages=985-94&amp;pmid=9545386&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R185">185. <span class="citation">Porter EA, Wang X, Lee HS, Weisblum B, Gellman SH. Non-haemolytic &#x003b2;-aminoacid oligomers. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">404</span>:565.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10766230" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Non-haemolytic+&#x003b2;-aminoacid+oligomers&amp;author=EA+Porter&amp;author=X+Wang&amp;author=HS+Lee&amp;author=B+Weisblum&amp;author=SH+Gellman&amp;volume=404&amp;publication_year=2000&amp;pages=565&amp;pmid=10766230&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R186">186. <span class="citation">Punta M, Rost B. Protein folding rates estimated from contact predictions. <span><span class="ref-journal">J Mol Biol. </span>2005;<span class="ref-vol">348</span>:507–12.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15826649" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Protein+folding+rates+estimated+from+contact+predictions&amp;author=M+Punta&amp;author=B+Rost&amp;volume=348&amp;publication_year=2005&amp;pages=507-12&amp;pmid=15826649&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R187">187. <span class="citation">Qiu L, Hagen SJ. A limiting speed for protein folding at low solvent viscosity. <span><span class="ref-journal">J Am Chem Soc. </span>2004;<span class="ref-vol">126</span>:3398–99.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15025447" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=A+limiting+speed+for+protein+folding+at+low+solvent+viscosity&amp;author=L+Qiu&amp;author=SJ+Hagen&amp;volume=126&amp;publication_year=2004&amp;pages=3398-99&amp;pmid=15025447&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R188">188. <span class="citation">Raleigh DP, Plaxco KW. The protein folding transition state: What are &#x003c6;-values really telling us? <span><span class="ref-journal">Protein Pept Lett. </span>2005;<span class="ref-vol">12</span>:117–22.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15723637" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Pept+Lett&amp;title=The+protein+folding+transition+state:+What+are+&#x003c6;-values+really+telling+us?&amp;author=DP+Raleigh&amp;author=KW+Plaxco&amp;volume=12&amp;publication_year=2005&amp;pages=117-22&amp;pmid=15723637&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R189">189. <span class="citation">Reich L, Weikl TR. Substructural cooperativity and parallel versus sequential events during protein unfolding. <span><span class="ref-journal">Proteins. </span>2006;<span class="ref-vol">63</span>:1052–58.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16544293" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Substructural+cooperativity+and+parallel+versus+sequential+events+during+protein+unfolding&amp;author=L+Reich&amp;author=TR+Weikl&amp;volume=63&amp;publication_year=2006&amp;pages=1052-58&amp;pmid=16544293&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R190">190. <span class="citation">Rost B, Eyrich VA. EVA: large-scale analysis of secondary structure prediction. <span><span class="ref-journal">Proteins. </span>2001;<span class="ref-vol">45</span>:192–99.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11835497" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=EVA:+large-scale+analysis+of+secondary+structure+prediction&amp;author=B+Rost&amp;author=VA+Eyrich&amp;volume=45&amp;publication_year=2001&amp;pages=192-99&amp;pmid=11835497&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R191">191. <span class="citation">Sadqi M, Fushman D, Munoz V. Atom-by-atom analysis of global downhill protein folding. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">442</span>:317–21.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16799571" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Atom-by-atom+analysis+of+global+downhill+protein+folding&amp;author=M+Sadqi&amp;author=D+Fushman&amp;author=V+Munoz&amp;volume=442&amp;publication_year=2006&amp;pages=317-21&amp;pmid=16799571&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R192">192. <span class="citation">Sali A, Blundell TL. Comparative protein modelling by satisfaction of spatial restraints. <span><span class="ref-journal">J Mol Biol. </span>1993;<span class="ref-vol">234</span>:779–815.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8254673" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&amp;author=A+Sali&amp;author=TL+Blundell&amp;volume=234&amp;publication_year=1993&amp;pages=779-815&amp;pmid=8254673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R193">193. <span class="citation">Sanchez IE, Kiefhaber T. Origin of unusual &#x003c6;-values in protein folding: evidence against specific nucleation sites. <span><span class="ref-journal">J Mol Biol. </span>2003;<span class="ref-vol">334</span>:1077–85.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14643667" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Origin+of+unusual+&#x003c6;-values+in+protein+folding:+evidence+against+specific+nucleation+sites&amp;author=IE+Sanchez&amp;author=T+Kiefhaber&amp;volume=334&amp;publication_year=2003&amp;pages=1077-85&amp;pmid=14643667&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R194">194. <span class="citation">Schuler B, Lipman EA, Eaton WA. Probing the free-energy surface for protein folding with single-molecule fluorescence spectroscopy. <span><span class="ref-journal">Nature. </span>2002;<span class="ref-vol">419</span>:743–47.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12384704" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Probing+the+free-energy+surface+for+protein+folding+with+single-molecule+fluorescence+spectroscopy&amp;author=B+Schuler&amp;author=EA+Lipman&amp;author=WA+Eaton&amp;volume=419&amp;publication_year=2002&amp;pages=743-47&amp;pmid=12384704&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R195">195. <span class="citation">Segel DJ, Bachmann A, Hofrichter J, Hodgson KO, Doniach S, Kiefhaber T. Characterization of transient intermediates in lysozyme folding with time-resolved small-angle X-ray scattering. <span><span class="ref-journal">J Mol Biol. </span>1999;<span class="ref-vol">288</span>:489–99.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10329156" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Characterization+of+transient+intermediates+in+lysozyme+folding+with+time-resolved+small-angle+X-ray+scattering&amp;author=DJ+Segel&amp;author=A+Bachmann&amp;author=J+Hofrichter&amp;author=KO+Hodgson&amp;author=S+Doniach&amp;volume=288&amp;publication_year=1999&amp;pages=489-99&amp;pmid=10329156&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R196">196. <span class="citation">Shea JE, Brooks CL., III From folding theories to folding proteins: a review and assessment of simulation studies of protein folding and unfolding. <span><span class="ref-journal">Annu Rev Phys Chem. </span>2001;<span class="ref-vol">52</span>:499–535.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11326073" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Phys+Chem&amp;title=From+folding+theories+to+folding+proteins:+a+review+and+assessment+of+simulation+studies+of+protein+folding+and+unfolding&amp;author=JE+Shea&amp;author=CL+Brooks&amp;volume=52&amp;publication_year=2001&amp;pages=499-535&amp;pmid=11326073&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R197">197. <span class="citation">Shea JE, Onuchic JN, Brooks CL. Exploring the origins of topological frustration: design of a minimally frustrated model of fragment B of protein A. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:12512–17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC22965/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10535953" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Exploring+the+origins+of+topological+frustration:+design+of+a+minimally+frustrated+model+of+fragment+B+of+protein+A&amp;author=JE+Shea&amp;author=JN+Onuchic&amp;author=CL+Brooks&amp;volume=96&amp;publication_year=1999&amp;pages=12512-17&amp;pmid=10535953&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R198">198. <span class="citation">Shirts M, Pande VS. Computing: Screen savers of the world unite! <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">290</span>:1903–4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17742054" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Computing:+Screen+savers+of+the+world+unite!&amp;author=M+Shirts&amp;author=VS+Pande&amp;volume=290&amp;publication_year=2000&amp;pages=1903-4&amp;pmid=17742054&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R199">199. <span class="citation">Shmygelska A. Search for folding nuclei in native protein structures. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:394–402.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15961484" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Search+for+folding+nuclei+in+native+protein+structures&amp;author=A+Shmygelska&amp;volume=21&amp;publication_year=2005&amp;pages=394-402&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R200">200. <span class="citation">Simmerling C, Strockbine B, Roitberg AE. All-atom structure prediction and folding simulations of a stable protein. <span><span class="ref-journal">J Am Chem Soc. </span>2002;<span class="ref-vol">124</span>:11258–59.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12236726" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=All-atom+structure+prediction+and+folding+simulations+of+a+stable+protein&amp;author=C+Simmerling&amp;author=B+Strockbine&amp;author=AE+Roitberg&amp;volume=124&amp;publication_year=2002&amp;pages=11258-59&amp;pmid=12236726&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R201">201. <span class="citation">Singhal N, Snow CD, Pande VS. Using path sampling to build better Markovian state models: predicting the folding rate and mechanism of a tryptophan zipper &#x003b2; hairpin. <span><span class="ref-journal">J Chem Phys. </span>2004;<span class="ref-vol">121</span>:415–25.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15260562" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Using+path+sampling+to+build+better+Markovian+state+models:+predicting+the+folding+rate+and+mechanism+of+a+tryptophan+zipper+&#x003b2;+hairpin&amp;author=N+Singhal&amp;author=CD+Snow&amp;author=VS+Pande&amp;volume=121&amp;publication_year=2004&amp;pages=415-25&amp;pmid=15260562&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R202">202. <span class="citation">Snow CD, Rhee YM, Pande VS. Kinetic definition of protein folding transition state ensembles and reaction coordinates. <span><span class="ref-journal">Biophys J. </span>2006;<span class="ref-vol">91</span>:14–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1479057/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16617068" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=Kinetic+definition+of+protein+folding+transition+state+ensembles+and+reaction+coordinates&amp;author=CD+Snow&amp;author=YM+Rhee&amp;author=VS+Pande&amp;volume=91&amp;publication_year=2006&amp;pages=14-24&amp;pmid=16617068&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R203">203. <span class="citation">Sohl JL, Jaswal SS, Agard DA. Unfolded conformations of &#x003b1;-lytic protease are more stable than its native state. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">395</span>:817–19.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9796818" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Unfolded+conformations+of+&#x003b1;-lytic+protease+are+more+stable+than+its+native+state&amp;author=JL+Sohl&amp;author=SS+Jaswal&amp;author=DA+Agard&amp;volume=395&amp;publication_year=1998&amp;pages=817-19&amp;pmid=9796818&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R204">204. <span class="citation">Sosnick TR, Dothager RS, Krantz BA. Differences in the folding transition state of ubiquitin indicated by &#x003c6; and &#x003c8; analyses. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2004;<span class="ref-vol">101</span>:17377–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC536030/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15576508" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Differences+in+the+folding+transition+state+of+ubiquitin+indicated+by+&#x003c6;+and+&#x003c8;+analyses&amp;author=TR+Sosnick&amp;author=RS+Dothager&amp;author=BA+Krantz&amp;volume=101&amp;publication_year=2004&amp;pages=17377-82&amp;pmid=15576508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R205">205. <span class="citation">Sridevi K, Juneja J, Bhuyan AK, Krishnamoorthy G, Udgaonkar JB. The slow folding reaction of barstar: The core tryptophan region attains tight packing before substantial secondary and tertiary structure formation and final compaction of the polypeptide chain. <span><span class="ref-journal">J Mol Biol. </span>2000;<span class="ref-vol">302</span>:479–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10970747" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=The+slow+folding+reaction+of+barstar:+The+core+tryptophan+region+attains+tight+packing+before+substantial+secondary+and+tertiary+structure+formation+and+final+compaction+of+the+polypeptide+chain&amp;author=K+Sridevi&amp;author=J+Juneja&amp;author=AK+Bhuyan&amp;author=G+Krishnamoorthy&amp;author=JB+Udgaonkar&amp;volume=302&amp;publication_year=2000&amp;pages=479-95&amp;pmid=10970747&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R206">206. <span class="citation">Sridevi K, Lakshmikanth GS, Krishnamoorthy G, Udgaonkar JB. Increasing stability reduces conformational heterogeneity in a protein folding intermediate ensemble. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">337</span>:699–711.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15019788" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Increasing+stability+reduces+conformational+heterogeneity+in+a+protein+folding+intermediate+ensemble&amp;author=K+Sridevi&amp;author=GS+Lakshmikanth&amp;author=G+Krishnamoorthy&amp;author=JB+Udgaonkar&amp;volume=337&amp;publication_year=2004&amp;pages=699-711&amp;pmid=15019788&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R207">207. <span class="citation">Srinivasan R, Rose GD. Ab initio prediction of protein structure using LINUS. <span><span class="ref-journal">Proteins. </span>2002;<span class="ref-vol">47</span>:489–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12001227" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Ab+initio+prediction+of+protein+structure+using+LINUS&amp;author=R+Srinivasan&amp;author=GD+Rose&amp;volume=47&amp;publication_year=2002&amp;pages=489-95&amp;pmid=12001227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R208">208. <span class="citation">Sriraman S, Kevrekidis IG, Hummer G. Coarse master equation from Bayesian analysis of replica molecular dynamics simulations. <span><span class="ref-journal">J Phys Chem B. </span>2005;<span class="ref-vol">109</span>:6479–84.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16851726" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Coarse+master+equation+from+Bayesian+analysis+of+replica+molecular+dynamics+simulations&amp;author=S+Sriraman&amp;author=IG+Kevrekidis&amp;author=G+Hummer&amp;volume=109&amp;publication_year=2005&amp;pages=6479-84&amp;pmid=16851726&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R209">209. <span class="citation">Street TO, Bradley CM, Barrick D. Predicting coupling limits from an experimentally determined energy landscape. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:4907–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1829238/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17360387" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Predicting+coupling+limits+from+an+experimentally+determined+energy+landscape&amp;author=TO+Street&amp;author=CM+Bradley&amp;author=D+Barrick&amp;volume=104&amp;publication_year=2007&amp;pages=4907-12&amp;pmid=17360387&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R210">210. <span class="citation">Sugita Y, Okamoto Y. Replica-exchange molecular dynamics method for protein folding. <span><span class="ref-journal">Chem Phys Lett. </span>1999;<span class="ref-vol">314</span>:141–51.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Phys+Lett&amp;title=Replica-exchange+molecular+dynamics+method+for+protein+folding&amp;author=Y+Sugita&amp;author=Y+Okamoto&amp;volume=314&amp;publication_year=1999&amp;pages=141-51&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R211">211. <span class="citation">Swope WC, Pitera JW, Suits F. Describing protein folding kinetics by molecular dynamics simulations. 1. Theory. <span><span class="ref-journal">J Phys Chem B. </span>2004;<span class="ref-vol">108</span>:6571–81.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Describing+protein+folding+kinetics+by+molecular+dynamics+simulations.+1.+Theory&amp;author=WC+Swope&amp;author=JW+Pitera&amp;author=F+Suits&amp;volume=108&amp;publication_year=2004&amp;pages=6571-81&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R212">212. <span class="citation">Swope WC, Pitera JW, Suits F, Pitman M, Eleftheriou M, et al. Describing protein folding kinetics by molecular dynamics simulations. 2. Example applications to alanine dipeptide and a B-hairpin peptide. <span><span class="ref-journal">J Phys Chem B. </span>2004;<span class="ref-vol">108</span>:6582–94.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Chem+B&amp;title=Describing+protein+folding+kinetics+by+molecular+dynamics+simulations.+2.+Example+applications+to+alanine+dipeptide+and+a+B-hairpin+peptide&amp;author=WC+Swope&amp;author=JW+Pitera&amp;author=F+Suits&amp;author=M+Pitman&amp;author=M+Eleftheriou&amp;volume=108&amp;publication_year=2004&amp;pages=6582-94&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R213">213. <span class="citation">Travaglini-Allocatelli C, Cutruzzol&#x000e0; F, Bigotti MG, Staniforth RA, Brunori M. Folding mechanism of <em>Pseudomonas aeruginosa</em> cytochrome <em>c</em>. <span><span class="ref-journal">J Mol Biol. </span>1999;<span class="ref-vol">289</span>:1459–67.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10373379" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Folding+mechanism+of+Pseudomonas+aeruginosa+cytochrome+c&amp;author=C+Travaglini-Allocatelli&amp;author=F+Cutruzzol&#x000e0;&amp;author=MG+Bigotti&amp;author=RA+Staniforth&amp;author=M+Brunori&amp;volume=289&amp;publication_year=1999&amp;pages=1459-67&amp;pmid=10373379&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R214">214. <span class="citation">Tress M, Ezkurdia I, Gra&#x000f1;a O, L&#x000f3;pez G, Valencia A. Assessment of predictions submitted for the CASP6 comparative modelling category. <span><span class="ref-journal">Proteins. </span>2005;<span class="ref-vol">61</span>:27–45.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16187345" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Assessment+of+predictions+submitted+for+the+CASP6+comparative+modelling+category&amp;author=M+Tress&amp;author=I+Ezkurdia&amp;author=O+Gra&#x000f1;a&amp;author=G+L&#x000f3;pez&amp;author=A+Valencia&amp;volume=61&amp;publication_year=2005&amp;pages=27-45&amp;pmid=16187345&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R215">215. <span class="citation">Tsai CJ, Maizel JV, Jr, Nussinov R. Anatomy of protein structures: visualizing how a one-dimensional protein chain folds into a three-dimensional shape. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2000;<span class="ref-vol">97</span>:12038–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC17290/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11050234" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Anatomy+of+protein+structures:+visualizing+how+a+one-dimensional+protein+chain+folds+into+a+three-dimensional+shape&amp;author=CJ+Tsai&amp;author=JV+Maizel&amp;author=R+Nussinov&amp;volume=97&amp;publication_year=2000&amp;pages=12038-43&amp;pmid=11050234&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R216">216. <span class="citation">Unger R, Moult J. Finding the lowest free energy conformation of a protein is an NP-hard problem: proof and implications. <span><span class="ref-journal">Bull Math Biol. </span>1993;<span class="ref-vol">55</span>:1183–98.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8281131" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bull+Math+Biol&amp;title=Finding+the+lowest+free+energy+conformation+of+a+protein+is+an+NP-hard+problem:+proof+and+implications&amp;author=R+Unger&amp;author=J+Moult&amp;volume=55&amp;publication_year=1993&amp;pages=1183-98&amp;pmid=8281131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R217">217. <span class="citation">Utku Y, Dehan E, Ouerfelli O, Piano F, Zuckermann RN, et al. A peptidomimetic siRNA transfection reagent for highly effective gene silencing. <span><span class="ref-journal">Mol Biosyst. </span>2006;<span class="ref-vol">2</span>:312–17.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16880950" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biosyst&amp;title=A+peptidomimetic+siRNA+transfection+reagent+for+highly+effective+gene+silencing&amp;author=Y+Utku&amp;author=E+Dehan&amp;author=O+Ouerfelli&amp;author=F+Piano&amp;author=RN+Zuckermann&amp;volume=2&amp;publication_year=2006&amp;pages=312-17&amp;pmid=16880950&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R218">218. <span class="citation">Uversky VN, Fink AL. The chicken-egg scenario of protein folding revisited. <span><span class="ref-journal">FEBS Lett. </span>2002;<span class="ref-vol">515</span>:79–83.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11943199" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=The+chicken-egg+scenario+of+protein+folding+revisited&amp;author=VN+Uversky&amp;author=AL+Fink&amp;volume=515&amp;publication_year=2002&amp;pages=79-83&amp;pmid=11943199&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R219">219. <span class="citation">Venclovas C, Zemla A, Fidelis K, Moult J. Assessment of progress over the CASP experiments. <span><span class="ref-journal">Proteins. </span>2003;<span class="ref-vol">53</span>:585–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14579350" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Assessment+of+progress+over+the+CASP+experiments&amp;author=C+Venclovas&amp;author=A+Zemla&amp;author=K+Fidelis&amp;author=J+Moult&amp;volume=53&amp;publication_year=2003&amp;pages=585-95&amp;pmid=14579350&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R220">220. <span class="citation">Verma A, Schug A, Lee KH, Wenzel W. Basin hopping simulations for all-atom protein folding. <span><span class="ref-journal">J Chem Phys. </span>2006;<span class="ref-vol">124</span>:044515.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16460193" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Phys&amp;title=Basin+hopping+simulations+for+all-atom+protein+folding&amp;author=A+Verma&amp;author=A+Schug&amp;author=KH+Lee&amp;author=W+Wenzel&amp;volume=124&amp;publication_year=2006&amp;pages=044515&amp;pmid=16460193&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R221">221. <span class="citation">Viguera AR, Serrano L, Wilmanns M. Different folding transition states may result in the same native structure. <span><span class="ref-journal">Nat Struct Biol. </span>1996;<span class="ref-vol">3</span>:874–80.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8836105" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Different+folding+transition+states+may+result+in+the+same+native+structure&amp;author=AR+Viguera&amp;author=L+Serrano&amp;author=M+Wilmanns&amp;volume=3&amp;publication_year=1996&amp;pages=874-80&amp;pmid=8836105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R222">222. <span class="citation">Vitkup D, Melamud E, Moult J, Sander C. Completeness in structural genomics. <span><span class="ref-journal">Nat Struct Biol. </span>2001;<span class="ref-vol">8</span>:559–66.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11373627" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Completeness+in+structural+genomics&amp;author=D+Vitkup&amp;author=E+Melamud&amp;author=J+Moult&amp;author=C+Sander&amp;volume=8&amp;publication_year=2001&amp;pages=559-66&amp;pmid=11373627&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R223">223. <span class="citation">Voelz VA, Dill KA. Exploring zipping and assembly as a protein folding principle. <span><span class="ref-journal">Proteins. </span>2006;<span class="ref-vol">66</span>:877–88.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17154424" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Exploring+zipping+and+assembly+as+a+protein+folding+principle&amp;author=VA+Voelz&amp;author=KA+Dill&amp;volume=66&amp;publication_year=2006&amp;pages=877-88&amp;pmid=17154424&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R224">224. <span class="citation">Wallace IM, Blackshields G, Higgins DG. Multiple sequence alignments. <span><span class="ref-journal">Curr Opin Struct Biol. </span>2005;<span class="ref-vol">15</span>:261–66.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15963889" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Multiple+sequence+alignments&amp;author=IM+Wallace&amp;author=G+Blackshields&amp;author=DG+Higgins&amp;volume=15&amp;publication_year=2005&amp;pages=261-66&amp;pmid=15963889&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R225">225. <span class="citation">Wallin S, Chan HS. Conformational entropic barriers in topology-dependent protein folding: perspectives from a simple native-centric polymer model. <span><span class="ref-journal">J Phys Condens Matter. </span>2006;<span class="ref-vol">18</span>:S307–28.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Phys+Condens+Matter&amp;title=Conformational+entropic+barriers+in+topology-dependent+protein+folding:+perspectives+from+a+simple+native-centric+polymer+model&amp;author=S+Wallin&amp;author=HS+Chan&amp;volume=18&amp;publication_year=2006&amp;pages=S307-28&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R226">226. <span class="citation">Wang L, Xie J, Schultz PG. Expanding the genetic code. <span><span class="ref-journal">Annu Rev Biophys Biomol Struct. </span>2006;<span class="ref-vol">35</span>:225–49.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16689635" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biophys+Biomol+Struct&amp;title=Expanding+the+genetic+code&amp;author=L+Wang&amp;author=J+Xie&amp;author=PG+Schultz&amp;volume=35&amp;publication_year=2006&amp;pages=225-49&amp;pmid=16689635&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R227">227. <span class="citation">Wang Z, Mottonen J, Goldsmith EJ. Kinetically controlled folding of the serpin plasminogen activator inhibitor 1. <span><span class="ref-journal">Biochemistry. </span>1996;<span class="ref-vol">35</span>:16443–48.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8987976" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Kinetically+controlled+folding+of+the+serpin+plasminogen+activator+inhibitor+1&amp;author=Z+Wang&amp;author=J+Mottonen&amp;author=EJ+Goldsmith&amp;volume=35&amp;publication_year=1996&amp;pages=16443-48&amp;pmid=8987976&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R228">228. <span class="citation">Weikl TR. Loop-closure events during protein folding: rationalizing the shape of &#x003c6;-value distributions. <span><span class="ref-journal">Proteins. </span>2005;<span class="ref-vol">60</span>:701–11.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16021610" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Loop-closure+events+during+protein+folding:+rationalizing+the+shape+of+&#x003c6;-value+distributions&amp;author=TR+Weikl&amp;volume=60&amp;publication_year=2005&amp;pages=701-11&amp;pmid=16021610&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R229">229. <span class="citation">Weikl TR, Dill KA. Folding kinetics of two-state proteins: effect of circularization, permutation, and crosslinks. <span><span class="ref-journal">J Mol Biol. </span>2003;<span class="ref-vol">332</span>:953–63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12972264" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Folding+kinetics+of+two-state+proteins:+effect+of+circularization,+permutation,+and+crosslinks&amp;author=TR+Weikl&amp;author=KA+Dill&amp;volume=332&amp;publication_year=2003&amp;pages=953-63&amp;pmid=12972264&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R230">230. <span class="citation">Weikl TR, Dill KA. Folding rates and low-entropy-loss routes of two-state proteins. <span><span class="ref-journal">J Mol Biol. </span>2003;<span class="ref-vol">329</span>:585–98.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12767836" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Folding+rates+and+low-entropy-loss+routes+of+two-state+proteins&amp;author=TR+Weikl&amp;author=KA+Dill&amp;volume=329&amp;publication_year=2003&amp;pages=585-98&amp;pmid=12767836&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R231">231. <span class="citation">Weikl TR, Dill KA. Transition-states in protein folding kinetics: the structural interpretation of &#x003c6; values. <span><span class="ref-journal">J Mol Biol. </span>2007;<span class="ref-vol">365</span>:1578–86.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17141267" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Transition-states+in+protein+folding+kinetics:+the+structural+interpretation+of+&#x003c6;+values&amp;author=TR+Weikl&amp;author=KA+Dill&amp;volume=365&amp;publication_year=2007&amp;pages=1578-86&amp;pmid=17141267&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R232">232. <span class="citation">Weikl TR, Palassini M, Dill KA. Cooperativity in two-state protein folding kinetics. <span><span class="ref-journal">Protein Sci. </span>2004;<span class="ref-vol">13</span>:822–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2286727/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14978313" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=Cooperativity+in+two-state+protein+folding+kinetics&amp;author=TR+Weikl&amp;author=M+Palassini&amp;author=KA+Dill&amp;volume=13&amp;publication_year=2004&amp;pages=822-29&amp;pmid=14978313&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R233">233. <span class="citation">White GWN, Gianni S, Grossmann JG, Jemth P, Fersht AR, Daggett V. Simulation and experiment conspire to reveal cryptic intermediates and a slide from the nucleation-condensation to framework mechanism of folding. <span><span class="ref-journal">J Mol Biol. </span>2005;<span class="ref-vol">350</span>:757–75.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15967458" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Simulation+and+experiment+conspire+to+reveal+cryptic+intermediates+and+a+slide+from+the+nucleation-condensation+to+framework+mechanism+of+folding&amp;author=GWN+White&amp;author=S+Gianni&amp;author=JG+Grossmann&amp;author=P+Jemth&amp;author=AR+Fersht&amp;volume=350&amp;publication_year=2005&amp;pages=757-75&amp;pmid=15967458&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R234">234. <span class="citation">Wolfenden R. Experimental measures of amino acid hydrophobicity and the topology of transmembrane and globular proteins. <span><span class="ref-journal">J Gen Physiol. </span>2007;<span class="ref-vol">129</span>:357–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2154378/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17438117" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Gen+Physiol&amp;title=Experimental+measures+of+amino+acid+hydrophobicity+and+the+topology+of+transmembrane+and+globular+proteins&amp;author=R+Wolfenden&amp;volume=129&amp;publication_year=2007&amp;pages=357-62&amp;pmid=17438117&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R235">235. <span class="citation">Wu CW, Seurynck SL, Lee KY, Barron AE. Helical peptoid mimics of lung surfactant protein C. <span><span class="ref-journal">Chem Biol. </span>2003;<span class="ref-vol">10</span>:1057–63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14652073" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Biol&amp;title=Helical+peptoid+mimics+of+lung+surfactant+protein+C&amp;author=CW+Wu&amp;author=SL+Seurynck&amp;author=KY+Lee&amp;author=AE+Barron&amp;volume=10&amp;publication_year=2003&amp;pages=1057-63&amp;pmid=14652073&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R236">236. <span class="citation">Wurth C, Kim W, Hecht MH. Combinatorial approaches to probe the sequence determinants of protein aggregation and amyloidogenicity. <span><span class="ref-journal">Protein Pept Lett. </span>2006;<span class="ref-vol">13</span>:279–86.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16515456" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Pept+Lett&amp;title=Combinatorial+approaches+to+probe+the+sequence+determinants+of+protein+aggregation+and+amyloidogenicity&amp;author=C+Wurth&amp;author=W+Kim&amp;author=MH+Hecht&amp;volume=13&amp;publication_year=2006&amp;pages=279-86&amp;pmid=16515456&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R237">237. <span class="citation">Xu Y, Purkayastha P, Gai F. Nanosecond folding dynamics of a three-stranded &#x003b2;-sheet. <span><span class="ref-journal">J Am Chem Soc. </span>2006;<span class="ref-vol">128</span>:15836–42.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17147395" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Nanosecond+folding+dynamics+of+a+three-stranded+&#x003b2;-sheet&amp;author=Y+Xu&amp;author=P+Purkayastha&amp;author=F+Gai&amp;volume=128&amp;publication_year=2006&amp;pages=15836-42&amp;pmid=17147395&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R238">238. <span class="citation">Yang JS, Chen WW, Skolnick J, Shakhnovich EI. All-atom ab initio folding of a diverse set of proteins. <span><span class="ref-journal">Structure. </span>2006;<span class="ref-vol">15</span>:53–63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17223532" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=All-atom+ab+initio+folding+of+a+diverse+set+of+proteins&amp;author=JS+Yang&amp;author=WW+Chen&amp;author=J+Skolnick&amp;author=EI+Shakhnovich&amp;volume=15&amp;publication_year=2006&amp;pages=53-63&amp;pmid=17223532&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R239">239. <span class="citation">Yeh SR, Rousseau DL. Hierarchical folding of cytochrome <em>c</em>. <span><span class="ref-journal">Nat Struct Biol. </span>2000;<span class="ref-vol">7</span>:443–45.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10881185" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Hierarchical+folding+of+cytochrome+c&amp;author=SR+Yeh&amp;author=DL+Rousseau&amp;volume=7&amp;publication_year=2000&amp;pages=443-45&amp;pmid=10881185&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R240">240. <span class="citation">Yoda T, Sugita Y, Okamoto Y. Secondary structure preferences of force fields for proteins evaluated by generalized-ensemble simulations. <span><span class="ref-journal">Chem Phys. </span>2004;<span class="ref-vol">307</span>:269–83.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Phys&amp;title=Secondary+structure+preferences+of+force+fields+for+proteins+evaluated+by+generalized-ensemble+simulations&amp;author=T+Yoda&amp;author=Y+Sugita&amp;author=Y+Okamoto&amp;volume=307&amp;publication_year=2004&amp;pages=269-83&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R241">241. <span class="citation">Zagrovic B, Snow CD, Shirts MR, Pande VS. Simulation of folding of a small &#x003b1;-helical protein in atomistic detail using worldwide-distributed computing. <span><span class="ref-journal">J Mol Biol. </span>2002;<span class="ref-vol">323</span>:927–37.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12417204" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Simulation+of+folding+of+a+small+&#x003b1;-helical+protein+in+atomistic+detail+using+worldwide-distributed+computing&amp;author=B+Zagrovic&amp;author=CD+Snow&amp;author=MR+Shirts&amp;author=VS+Pande&amp;volume=323&amp;publication_year=2002&amp;pages=927-37&amp;pmid=12417204&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R242">242. <span class="citation">Zhang Y, Arakaki AK, Skolnick J. TASSER: an automated method for the prediction of protein tertiary structures in CASP6. <span><span class="ref-journal">Proteins. </span>2005;<span class="ref-vol">61</span>:91–98.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16187349" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=TASSER:+an+automated+method+for+the+prediction+of+protein+tertiary+structures+in+CASP6&amp;author=Y+Zhang&amp;author=AK+Arakaki&amp;author=J+Skolnick&amp;volume=61&amp;publication_year=2005&amp;pages=91-98&amp;pmid=16187349&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R243">243. <span class="citation">Zhao H, Giver L, Shao Z, Affholter JA, Arnold FH. Molecular evolution by staggered extension process (StEP) in vitro recombination. <span><span class="ref-journal">Nat Biotechnol. </span>1998;<span class="ref-vol">16</span>:258–61.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9528005" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Molecular+evolution+by+staggered+extension+process+(StEP)+in+vitro+recombination&amp;author=H+Zhao&amp;author=L+Giver&amp;author=Z+Shao&amp;author=JA+Affholter&amp;author=FH+Arnold&amp;volume=16&amp;publication_year=1998&amp;pages=258-61&amp;pmid=9528005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R244">244. <span class="citation">Zhong S, Rousseau DL, Yeh SR. Modulation of the folding energy landscape of cytochrome <em>c</em> with salt. <span><span class="ref-journal">J Am Chem Soc. </span>2004;<span class="ref-vol">126</span>:13934–35.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15506749" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Chem+Soc&amp;title=Modulation+of+the+folding+energy+landscape+of+cytochrome+c+with+salt&amp;author=S+Zhong&amp;author=DL+Rousseau&amp;author=SR+Yeh&amp;volume=126&amp;publication_year=2004&amp;pages=13934-35&amp;pmid=15506749&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R245">245. <span class="citation">Zhou HY, Zhou YQ. Folding rate prediction using total contact distance. <span><span class="ref-journal">Biophys J. </span>2002;<span class="ref-vol">82</span>:458–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1302485/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11751332" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys+J&amp;title=Folding+rate+prediction+using+total+contact+distance&amp;author=HY+Zhou&amp;author=YQ+Zhou&amp;volume=82&amp;publication_year=2002&amp;pages=458-63&amp;pmid=11751332&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R246">246. <span class="citation">Zhou R. Free energy landscape of protein folding in water: explicit vs. implicit solvent. <span><span class="ref-journal">Proteins. </span>2003;<span class="ref-vol">53</span>:148–61.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14517967" ref="reftype=pubmed&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Free+energy+landscape+of+protein+folding+in+water:+explicit+vs.+implicit+solvent&amp;author=R+Zhou&amp;volume=53&amp;publication_year=2003&amp;pages=148-61&amp;pmid=14517967&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=2443096&amp;issue-id=168275&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC2443096/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC2443096/pdf/nihms51627.pdf" class="int-view">PDF (2.0M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/2443096/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/2443096/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<div class="collections-button-container" data-article-id="2443096" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="zAMJXfQ8hT1ArsbvQwKKC3kepYqjt3YQLgbn6cO4aGwPhbwc6daa7ccNiFr4VvGL">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F&amp;text=The%20Protein%20Folding%20Problem" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2443096%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2443096/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/18573083/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/18573083/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC2443096/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/2443096/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.54110350c77632754ed7.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'" /><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css" /><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.586993420cfd.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.bf4d0b9d90eb88f67344.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC2443096/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
